apis/sheets/v4.js

```diff
@@ -131,6 +131,38 @@ function Sheets(options) { // eslint-disable-line
     values: {
 
       /**
+       * sheets.spreadsheets.values.append
+       *
+       * @desc Appends values to a spreadsheet. The input range is used to search for existing data and find a "table" within that range. Values will be appended to the next row of the table, starting with the first column of the table. See the [guide](/sheets/guides/values#appending_values) and [sample code](/sheets/samples/writing#append_values) for specific details of how tables are detected and data is appended.  The caller must specify the spreadsheet ID, range, and a valueInputOption.  The `valueInputOption` only controls how the input data will be added to the sheet (column-wise or row-wise), it does not influence what cell the data starts being written to.
+       *
+       * @alias sheets.spreadsheets.values.append
+       * @memberOf! sheets(v4)
+       *
+       * @param {object} params Parameters for request
+       * @param {string=} params.valueInputOption How the input data should be interpreted.
+       * @param {string} params.spreadsheetId The ID of the spreadsheet to update.
+       * @param {string=} params.insertDataOption How the input data should be inserted.
+       * @param {string} params.range The A1 notation of a range to search for a logical table of data. Values will be appended after the last row of the table.
+       * @param {object} params.resource Request body data
+       * @param {callback} callback The callback that handles the response.
+       * @return {object} Request object
+       */
+      append: function (params, callback) {
+        var parameters = {
```

Comment:
Stop all of this 

before_mach:60221306321;after_mach:60221306505;rtc_s:8653862.639968958;type:11;t_ns:491529894262661628
2016-07-29 20:04:54 -0400 timed[62]: t_s:491529894.2625939;rateSf:0.9999887129874205;sf_var:2.968093325301976e-11;rtc_s:8653862.639968958;utc_var:8.391809525498135e-08;type:29;utc_sf_cov:9.469499253300751e-11
2016-07-29 20:04:54 -0400 timed[62]: _utc_sf_cov:7.096764354110316e-11;_rateSf:0.9999887321322736;_rtc:8653862.639968958;_utc_var:6.28910707039549e-08;type:32;_utc:491529894.2626109;_sf_var:2.965415879956262e-11
2016-07-29 20:04:54 -0400 timed[62]: type:21;rtc_s:8653862.673919;mach:60222121861
2016-07-29 20:04:54 -0400 timed[62]: need_rtc_s:9153850.100922871;bb_want_rtc_s:inf;type:20;rtc_s:8653862.685560999;want_rtc_s:8656350.100922871
2016-07-29 20:04:54 -0400 timed[62]: t_s:491529894.2940615;adjust:-0.001323461532592773;didSet:1;mach:60222061884;rtc_s:8653862.671419958;unc_s:0.0002507932504009432;after_mach:60222062035;type:3;settimeofday:0
2016-07-29 20:19:43 -0400 timed[62]: type:21;rtc_s:8654751.960387999;mach:63054534257
2016-07-29 20:19:43 -0400 timed[62]: type:22;mach:63045159002
2016-07-29 20:19:43 -0400 timed[62]: type:9;src:10;rtc_s:8654751.958419001
2016-07-29 20:45:15 -0400 timed[62]: type:21;rtc_s:8656284.305877;mach:64774217856
2016-07-29 20:45:15 -0400 timed[62]: type:22;mach:64764968463
2016-07-29 20:45:15 -0400 timed[62]: type:9;src:10;rtc_s:8656284.301772
2016-07-29 20:50:18 -0400 timed[62]: type:22;mach:64971535341
2016-07-29 20:50:18 -0400 timed[62]: port:123;ipv4:301798525;delay:0.03408222924917936;mach:65014635290;type:2;tv_usec:374781;dispersion:0.0009918212890625;attempts:0;num:0;tv_sec:1469839818;result:0;slept:0;more:1
2016-07-29 20:50:18 -0400 timed[62]: type:21;rtc_s:8656586.787109001;mach:65014730007
2016-07-29 20:50:18 -0400 timed[62]: port:123;ipv4:301798653;delay:0.03602567873895168;mach:65015515511;type:2;tv_usec:412557;dispersion:0.0010986328125;attempts:0;num:1;tv_sec:1469839818;result:0;slept:0;more:1
2016-07-29 20:50:18 -0400 timed[62]: type:21;rtc_s:8656586.821472;mach:65015558174
2016-07-29 20:50:18 -0400 timed[62]: port:123;ipv4:301799549;delay:0.05368417385034263;mach:65016818386;type:2;tv_usec:466056;dispersion:0.0009918212890625;attempts:0;num:2;tv_sec:1469839818;result:0;slept:0;more:0
2016-07-29 20:50:18 -0400 timed[62]: type:21;rtc_s:8656586.875335;mach:65016847175
2016-07-29 20:50:18 -0400 timed[62]: mach:65014635290;unc_s:0.01853180209309855;t_s:491532618.3753015;type:5;src:5;rtc_s:8656586.783018501
2016-07-29 20:50:18 -0400 timed[62]: t_s:491532618.3749652;rateSf:0.9999887321322736;sf_var:1.053853355768781e-10;rtc_s:8656586.783018501;utc_var:0.0004078442922193049;type:29;utc_sf_cov:1.84004417653281e-07
2016-07-29 20:50:18 -0400 timed[62]: _utc_sf_cov:8.411362792965342e-08;_rateSf:0.9999888144975658;_rtc:8656586.783018501;_utc_var:0.0001864371708379782;type:32;_utc:491532618.3751478;_sf_var:6.03182672307579e-11
2016-07-29 20:50:18 -0400 timed[62]: type:21;rtc_s:8656586.879608;mach:65016951853
2016-07-29 20:50:18 -0400 timed[62]: need_rtc_s:9155904.073030956;bb_want_rtc_s:inf;type:20;rtc_s:8656586.896774;want_rtc_s:8658404.073030956
2016-07-29 20:50:18 -0400 timed[62]: t_s:491532618.4712273;adjust:-0.03029769659042358;didSet:1;mach:65016939640;rtc_s:8656586.879099125;unc_s:0.01365479369154734;after_mach:65016939659;type:3;settimeofday:0
2016-07-29 20:57:42 -0400 timed[62]: type:21;rtc_s:8657030.735992;mach:65611860462
2016-07-29 20:57:42 -0400 timed[62]: type:22;mach:65383380138
2016-07-29 20:57:42 -0400 timed[62]: unc_ns:500919;src:14;before_mach:65611235917;after_mach:65611236050;rtc_s:8657030.709969291;type:11;t_ns:491533062296748255
2016-07-29 20:57:42 -0400 timed[62]: t_s:491533062.297133;rateSf:0.9999888144975658;sf_var:7.265943646271956e-11;rtc_s:8657030.709969291;utc_var:0.0002738154692016415;type:29;utc_sf_cov:1.136298211915034e-07
2016-07-29 20:57:42 -0400 timed[62]: _utc_sf_cov:1.040332921687757e-10;_rateSf:0.9999886549783906;_rtc:8657030.709969291;_utc_var:2.506905705657733e-07;type:32;_utc:491533062.2967486;_sf_var:2.55477264587597e-11
2016-07-29 20:57:42 -0400 timed[62]: need_rtc_s:9157005.675464574;bb_want_rtc_s:inf;type:20;rtc_s:8657030.758041;want_rtc_s:8659505.675464574
2016-07-29 20:57:42 -0400 timed[62]: type:21;rtc_s:8657030.761413001;mach:65612467306
2016-07-29 20:57:42 -0400 timed[62]: t_s:491533062.3275851;adjust:-0.005145907402038574;didSet:1;mach:65611972740;rtc_s:8657030.740806084;unc_s:0.0005006996940049856;after_mach:65611972763;type:3;settimeofday:0
2016-07-29 20:57:43 -0400 timed[62]: type:21;rtc_s:8657031.855438;mach:65638726879
2016-07-29 20:57:43 -0400 timed[62]: unc_ns:500959;src:13;before_mach:65638054065;after_mach:65638054234;rtc_s:8657031.827404084;type:11;t_ns:491533063414155819
2016-07-29 20:57:43 -0400 timed[62]: t_s:491533063.4141707;rateSf:0.9999886549783906;sf_var:2.557879114598344e-11;rtc_s:8657031.827404084;utc_var:2.509549847732336e-07;type:29;utc_sf_cov:1.32598566957085e-10
2016-07-29 20:57:43 -0400 timed[62]: _utc_sf_cov:6.630002511750226e-11;_rateSf:0.9999886510417274;_rtc:8657031.827404084;_utc_var:1.254788960065662e-07;type:32;_utc:491533063.4141633;_sf_var:2.554376059400131e-11
2016-07-29 20:57:43 -0400 timed[62]: type:21;rtc_s:8657031.859436;mach:65638821253
2016-07-29 20:57:43 -0400 timed[62]: need_rtc_s:9157014.115903864;bb_want_rtc_s:inf;type:20;rtc_s:8657031.872145999;want_rtc_s:8659514.115903864
2016-07-29 20:57:43 -0400 timed[62]: t_s:491533063.4441673;adjust:-0.001118719577789307;didSet:1;mach:65638772591;rtc_s:8657031.857408416;unc_s:0.0003542405342556131;after_mach:65638773141;type:3;settimeofday:0
2016-07-29 20:57:44 -0400 timed[62]: type:21;rtc_s:8657032.981201001;mach:65665745525
2016-07-29 20:57:44 -0400 timed[62]: unc_ns:500975;src:13;before_mach:65665054235;after_mach:65665054404;rtc_s:8657032.95239725;type:11;t_ns:491533064539145231
2016-07-29 20:57:44 -0400 timed[62]: t_s:491533064.5391437;rateSf:0.9999886510417274;sf_var:2.557503540401485e-11;rtc_s:8657032.95239725;utc_var:1.256604117800128e-07;type:29;utc_sf_cov:9.505417319230892e-11
2016-07-29 20:57:44 -0400 timed[62]: _utc_sf_cov:6.334044251236566e-11;_rateSf:0.9999886514328404;_rtc:8657032.95239725;_utc_var:8.37352619156348e-08;type:32;_utc:491533064.5391442;_sf_var:2.555104596767668e-11
2016-07-29 20:57:44 -0400 timed[62]: type:21;rtc_s:8657032.986083001;mach:65665865030
2016-07-29 20:57:44 -0400 timed[62]: need_rtc_s:9157018.483874355;bb_want_rtc_s:inf;type:20;rtc_s:8657032.999190999;want_rtc_s:8659518.483874355
2016-07-29 20:57:44 -0400 timed[62]: t_s:491533064.5705182;adjust:0.0001041889190673828;didSet:1;mach:65665809556;rtc_s:8657032.983771585;unc_s:0.0002893778155235147;after_mach:65665809576;type:3;settimeofday:0
2016-07-29 20:57:45 -0400 timed[62]: type:21;rtc_s:8657034.106535999;mach:65692753224
2016-07-29 20:57:45 -0400 timed[62]: unc_ns:500941;src:13;before_mach:65692102367;after_mach:65692102520;rtc_s:8657034.079416957;type:11;t_ns:491533065666134181
2016-07-29 20:57:45 -0400 timed[62]: t_s:491533065.6661511;rateSf:0.9999886514328404;sf_var:2.558237711553443e-11;rtc_s:8657034.079416957;utc_var:8.391050129434967e-08;type:29;utc_sf_cov:9.21546302658414e-11
2016-07-29 20:57:45 -0400 timed[62]: _utc_sf_cov:6.90616713487924e-11;_rateSf:0.999988646774173;_rtc:8657034.079416957;_utc_var:6.288343240470692e-08;type:32;_utc:491533065.6661469;_sf_var:2.555701529480976e-11
2016-07-29 20:57:45 -0400 timed[62]: type:21;rtc_s:8657034.119720001;mach:65693074327
2016-07-29 20:57:45 -0400 timed[62]: need_rtc_s:9157021.541132344;bb_want_rtc_s:inf;type:20;rtc_s:8657034.133651;want_rtc_s:8659521.541132344
2016-07-29 20:57:45 -0400 timed[62]: t_s:491533065.7013127;adjust:-0.0001003146171569824;didSet:1;mach:65692951043;rtc_s:8657034.114583168;unc_s:0.0002507771805527679;after_mach:65692951065;type:3;settimeofday:0
2016-07-29 20:57:46 -0400 timed[62]: type:21;rtc_s:8657035.17633;mach:65718428927
2016-07-29 20:57:46 -0400 timed[62]: unc_ns:500931;src:13;before_mach:65717789797;after_mach:65717789937;rtc_s:8657035.149699584;type:11;t_ns:491533066736427547
2016-07-29 20:57:46 -0400 timed[62]: t_s:491533066.7364174;rateSf:0.999988646774173;sf_var:2.55867691518375e-11;rtc_s:8657035.149699584;utc_var:6.306055046611444e-08;type:29;utc_sf_cov:9.643082333230064e-11
2016-07-29 20:57:46 -0400 timed[62]: _utc_sf_cov:7.706421234329065e-11;_rateSf:0.999988649904373;_rtc:8657035.149699584;_utc_var:5.039583282265331e-08;type:32;_utc:491533066.7364194;_sf_var:2.555715415119093e-11
2016-07-29 20:57:46 -0400 timed[62]: need_rtc_s:9157023.925191484;bb_want_rtc_s:inf;type:20;rtc_s:8657035.202071;want_rtc_s:8659523.925191484
2016-07-29 20:57:46 -0400 timed[62]: type:21;rtc_s:8657035.20465;mach:65719105950
2016-07-29 20:57:46 -0400 timed[62]: t_s:491533066.7708933;adjust:0.0002692937850952148;didSet:1;mach:65718614524;rtc_s:8657035.184173916;unc_s:0.0002245065823374465;after_mach:65718614543;type:3;settimeofday:0
2016-07-29 20:57:47 -0400 timed[62]: type:21;rtc_s:8657036.282867;mach:65744982550
2016-07-29 20:57:47 -0400 timed[62]: unc_ns:500878;src:13;before_mach:65744407439;after_mach:65744407551;rtc_s:8657036.258904042;type:11;t_ns:491533067845477682
2016-07-29 20:57:47 -0400 timed[62]: t_s:491533067.8456113;rateSf:0.999988649904373;sf_var:2.558799003512536e-11;rtc_s:8657036.258904042;utc_var:5.059824925381817e-08;type:29;utc_sf_cov:1.054294233134093e-10
2016-07-29 20:57:47 -0400 timed[62]: _utc_sf_cov:8.773474249450601e-11;_rateSf:0.9999886031922932;_rtc:8657036.258904042;_utc_var:4.210612397793468e-08;type:32;_utc:491533067.8455889;_sf_var:2.555112038010837e-11
2016-07-29 20:57:47 -0400 timed[62]: type:21;rtc_s:8657036.285429999;mach:65745043694
2016-07-29 20:57:47 -0400 timed[62]: need_rtc_s:9157025.999015644;bb_want_rtc_s:inf;type:20;rtc_s:8657036.294050999;want_rtc_s:8659525.999015644
2016-07-29 20:57:47 -0400 timed[62]: t_s:491533067.8705931;adjust:6.139278411865234e-06;didSet:1;mach:65745007181;rtc_s:8657036.283908624;unc_s:0.0002052118757658364;after_mach:65745007189;type:3;settimeofday:0
2016-07-29 20:57:48 -0400 timed[62]: type:21;rtc_s:8657037.388458;mach:65771517165
2016-07-29 20:57:48 -0400 timed[62]: unc_ns:500983;src:13;before_mach:65770816111;after_mach:65770816263;rtc_s:8657037.359247416;type:11;t_ns:491533068945825154
2016-07-29 20:57:48 -0400 timed[62]: t_s:491533068.9459197;rateSf:0.9999886031922932;sf_var:2.558170992592046e-11;rtc_s:8657037.359247416;utc_var:4.233014916926442e-08;type:29;utc_sf_cov:1.158665780187479e-10
2016-07-29 20:57:48 -0400 timed[62]: _utc_sf_cov:9.914509846341202e-11;_rateSf:0.9999885658493805;_rtc:8657037.359247416;_utc_var:3.622120268951558e-08;type:32;_utc:491533068.945906;_sf_var:2.553593969705261e-11
2016-07-29 20:57:48 -0400 timed[62]: need_rtc_s:9157027.843313057;bb_want_rtc_s:inf;type:20;rtc_s:8657037.408645;want_rtc_s:8659527.843313057
2016-07-29 20:57:49 -0400 timed[62]: type:21;rtc_s:8657037.411071001;mach:65772059458
2016-07-29 20:57:49 -0400 timed[62]: t_s:491533068.9779815;adjust:-4.249811172485352e-05;didSet:1;mach:65771585512;rtc_s:8657037.39132325;unc_s:0.0001903392675850352;after_mach:65771585534;type:3;settimeofday:0
2016-07-29 21:13:35 -0400 timed[62]: type:21;rtc_s:8657983.781982001;mach:66706342009
2016-07-29 21:13:35 -0400 timed[62]: type:22;mach:66696295981
2016-07-29 21:13:35 -0400 timed[62]: type:9;src:10;rtc_s:8657983.721419999
2016-07-29 21:41:20 -0400 timed[62]: type:21;rtc_s:8659649.378997;mach:67779093038
2016-07-29 21:41:20 -0400 timed[62]: type:22;mach:67770665255
2016-07-29 21:41:20 -0400 timed[62]: type:9;src:10;rtc_s:8659649.373794001
2016-07-29 21:41:22 -0400 timed[62]: port:123;ipv4:301798653;delay:0.1656179712153971;mach:67823611470;type:2;tv_usec:731349;dispersion:0.0009918212890625;attempts:0;num:0;tv_sec:1469842882;result:0;slept:0;more:1
2016-07-29 21:41:22 -0400 timed[62]: type:21;rtc_s:8659651.23587;mach:67823658602
2016-07-29 21:41:23 -0400 timed[62]: port:123;ipv4:301799549;delay:0.2297166525386274;mach:67829141303;type:2;tv_usec:20300;dispersion:0.0009918212890625;attempts:0;num:1;tv_sec:1469842883;result:0;slept:0;more:1
2016-07-29 21:41:23 -0400 timed[62]: type:21;rtc_s:8659651.466033;mach:67829182909
2016-07-29 21:41:23 -0400 timed[62]: port:123;ipv4:301798525;delay:0.1194052658975124;mach:67832023491;type:2;tv_usec:108909;dispersion:0.001007080078125;attempts:0;num:2;tv_sec:1469842883;result:0;slept:0;more:0
2016-07-29 21:41:23 -0400 timed[62]: type:21;rtc_s:8659651.585875999;mach:67832058589
2016-07-29 21:41:23 -0400 timed[62]: mach:67823611470;unc_s:0.05924738410204366;t_s:491535682.7569458;type:5;src:5;rtc_s:8659651.233889708
2016-07-29 21:41:23 -0400 timed[62]: t_s:491535682.7906609;rateSf:0.9999885658493805;sf_var:9.820165475277126e-11;rtc_s:8659651.233889708;utc_var:0.0003405170718091351;type:29;utc_sf_cov:1.618164253137687e-07
2016-07-29 21:41:23 -0400 timed[62]: _utc_sf_cov:1.475072712699454e-07;_rateSf:0.9999871490776149;_rtc:8659651.233889708;_utc_var:0.000310405720469983;type:32;_utc:491535682.7876796;_sf_var:9.14018307547152e-11
2016-07-29 21:41:23 -0400 timed[62]: type:21;rtc_s:8659651.592559;mach:67832217935
2016-07-29 21:41:23 -0400 timed[62]: need_rtc_s:9158770.317150638;bb_want_rtc_s:inf;type:20;rtc_s:8659651.608076999;want_rtc_s:8661270.317150638
2016-07-29 21:41:23 -0400 timed[62]: t_s:491535683.1440507;adjust:-0.03268134593963623;didSet:1;mach:67832162889;rtc_s:8659651.590265417;unc_s:0.01762132188718331;after_mach:67832162907;type:3;settimeofday:0
2016-07-29 22:09:12 -0400 timed[62]: type:21;rtc_s:8661320.911436999;mach:69279218164
2016-07-29 22:09:12 -0400 timed[62]: type:22;mach:69269996075
2016-07-29 22:09:12 -0400 timed[62]: type:9;src:10;rtc_s:8661320.908553001
2016-07-29 22:09:12 -0400 timed[62]: port:123;ipv4:301798525;delay:0.03414012095890939;mach:69287974280;type:2;tv_usec:814884;dispersion:0.0010833740234375;attempts:0;num:0;tv_sec:1469844552;result:0;slept:0;more:1
2016-07-29 22:09:12 -0400 timed[62]: type:21;rtc_s:8661321.280731;mach:69288081013
2016-07-29 22:09:12 -0400 timed[62]: port:123;ipv4:301798653;delay:0.03472085692919791;mach:69288821605;type:2;tv_usec:850489;dispersion:0.0010833740234375;attempts:0;num:1;tv_sec:1469844552;result:0;slept:0;more:1
2016-07-29 22:09:12 -0400 timed[62]: type:21;rtc_s:8661321.314025;mach:69288880808
2016-07-29 22:09:12 -0400 timed[62]: port:123;ipv4:301799549;delay:0.05270679900422692;mach:69290101029;type:2;tv_usec:904147;dispersion:0.0010833740234375;attempts:0;num:2;tv_sec:1469844552;result:0;slept:0;more:0
2016-07-29 22:09:12 -0400 timed[62]: type:21;rtc_s:8661321.366087999;mach:69290129407
2016-07-29 22:09:12 -0400 timed[62]: mach:69287974280;unc_s:0.01865136316526815;t_s:491537352.8153914;type:5;src:5;rtc_s:8661321.276253
2016-07-29 22:09:12 -0400 timed[62]: t_s:491537352.8085812;rateSf:0.9999871490776149;sf_var:1.378290084542219e-10;rtc_s:8661321.276253;utc_var:0.001101178444712608;type:29;utc_sf_cov:3.389198774958512e-07
2016-07-29 22:09:12 -0400 timed[62]: _utc_sf_cov:8.136437431792443e-08;_rateSf:0.9999887419040361;_rtc:8661321.276253;_utc_var:0.0002643595171473036;type:32;_utc:491537352.8137565;_sf_var:5.855876846307596e-11
2016-07-29 22:09:12 -0400 timed[62]: type:21;rtc_s:8661321.380065;mach:69290464587
2016-07-29 22:09:12 -0400 timed[62]: need_rtc_s:9160508.319431964;bb_want_rtc_s:inf;type:20;rtc_s:8661321.393478001;want_rtc_s:8663008.319431964
2016-07-29 22:09:13 -0400 timed[62]: t_s:491537352.9167165;adjust:-0.01604348421096802;didSet:1;mach:69290444167;rtc_s:8661321.379214166;unc_s:0.01625965927771023;after_mach:69290444188;type:3;settimeofday:0
2016-07-29 22:24:07 -0400 timed[62]: type:21;rtc_s:8662215.924011;mach:73488231404
2016-07-29 22:24:07 -0400 timed[62]: type:22;mach:73226720004
2016-07-29 22:24:07 -0400 timed[62]: unc_ns:500955;src:13;before_mach:73487563348;after_mach:73487563526;rtc_s:8662215.896175332;type:11;t_ns:491538247424360804
2016-07-29 22:24:07 -0400 timed[62]: t_s:491538247.4236071;rateSf:0.9999887419040361;sf_var:8.342920230391851e-11;rtc_s:8662215.896175332;utc_var:0.0004634420779430138;type:29;utc_sf_cov:1.448770080077809e-07
2016-07-29 22:24:07 -0400 timed[62]: _utc_sf_cov:7.840929776494383e-11;_rateSf:0.9999889773915225;_rtc:8662215.896175332;_utc_var:2.508208057712405e-07;type:32;_utc:491538247.4243604;_sf_var:3.81635936264821e-11
2016-07-29 22:24:07 -0400 timed[62]: type:21;rtc_s:8662215.927825;mach:73488322243
2016-07-29 22:24:07 -0400 timed[62]: need_rtc_s:9162190.855168674;bb_want_rtc_s:inf;type:20;rtc_s:8662215.940810001;want_rtc_s:8664690.855168674
2016-07-29 22:24:07 -0400 timed[62]: t_s:491538247.4543984;adjust:-0.009283602237701416;didSet:1;mach:73488283571;rtc_s:8662215.926213667;unc_s:0.0005008289312041528;after_mach:73488283592;type:3;settimeofday:0
2016-07-29 22:24:08 -0400 timed[62]: type:21;rtc_s:8662216.992064999;mach:73513861680
2016-07-29 22:24:08 -0400 timed[62]: unc_ns:500937;src:13;before_mach:73513215597;after_mach:73513215773;rtc_s:8662216.965144875;type:11;t_ns:491538248493188885
2016-07-29 22:24:08 -0400 timed[62]: t_s:491538248.4933181;rateSf:0.9999889773915225;sf_var:3.819331097975416e-11;rtc_s:8662216.965144875;utc_var:2.510320607740873e-07;type:29;utc_sf_cov:1.192209004438095e-10
2016-07-29 22:24:08 -0400 timed[62]: _utc_sf_cov:5.959931362524241e-11;_rateSf:0.9999889466861765;_rtc:8662216.965144875;_utc_var:1.254925811193419e-07;type:32;_utc:491538248.4932535;_sf_var:3.816499531662939e-11
2016-07-29 22:24:08 -0400 timed[62]: type:21;rtc_s:8662216.998717999;mach:73514022315
2016-07-29 22:24:08 -0400 timed[62]: need_rtc_s:9162199.252678847;bb_want_rtc_s:inf;type:20;rtc_s:8662217.012709999;want_rtc_s:8664699.252678847
2016-07-29 22:24:08 -0400 timed[62]: t_s:491538248.5231591;adjust:-0.005346834659576416;didSet:1;mach:73513934304;rtc_s:8662216.995050874;unc_s:0.0003542582443527569;after_mach:73513934393;type:3;settimeofday:0
2016-07-29 22:24:09 -0400 timed[62]: type:21;rtc_s:8662218.080108;mach:73539975285
2016-07-29 22:24:09 -0400 timed[62]: unc_ns:500946;src:13;before_mach:73539318489;after_mach:73539318660;rtc_s:8662218.0527415;type:11;t_ns:491538249580793297
2016-07-29 22:24:09 -0400 timed[62]: t_s:491538249.5808381;rateSf:0.9999889466861765;sf_var:3.819523050280688e-11;rtc_s:8662218.0527415;utc_var:1.25667377156788e-07;type:29;utc_sf_cov:1.011238755713934e-10
2016-07-29 22:24:09 -0400 timed[62]: _utc_sf_cov:6.738123224324359e-11;_rateSf:0.999988934666948;_rtc:8662218.0527415;_utc_var:8.373514837870983e-08;type:32;_utc:491538249.5808231;_sf_var:3.81680779756947e-11
2016-07-29 22:24:09 -0400 timed[62]: type:21;rtc_s:8662218.087432001;mach:73540150561
2016-07-29 22:24:09 -0400 timed[62]: need_rtc_s:9162203.584228413;bb_want_rtc_s:inf;type:20;rtc_s:8662218.099593;want_rtc_s:8664703.584228413
2016-07-29 22:24:09 -0400 timed[62]: t_s:491538249.6112062;adjust:-0.001016855239868164;didSet:1;mach:73540047191;rtc_s:8662218.083124917;unc_s:0.0002893818425112317;after_mach:73540047283;type:3;settimeofday:0
2016-07-29 22:24:10 -0400 timed[62]: type:21;rtc_s:8662219.124572;mach:73565045036
2016-07-29 22:24:10 -0400 timed[62]: unc_ns:500937;src:13;before_mach:73564399111;after_mach:73564399287;rtc_s:8662219.097658457;type:11;t_ns:491538250625803999
2016-07-29 22:24:10 -0400 timed[62]: t_s:491538250.6257285;rateSf:0.999988934666948;sf_var:3.819712666711645e-11;rtc_s:8662219.097658457;utc_var:8.391764840660442e-08;type:29;utc_sf_cov:1.072788808948191e-10
2016-07-29 22:24:10 -0400 timed[62]: _utc_sf_cov:8.039390094292382e-11;_rateSf:0.9999889588421562;_rtc:8662219.097658457;_utc_var:6.288718764672979e-08;type:32;_utc:491538250.6257474;_sf_var:3.816275737034893e-11
2016-07-29 22:24:10 -0400 timed[62]: need_rtc_s:9162206.558999473;bb_want_rtc_s:inf;type:20;rtc_s:8662219.144362999;want_rtc_s:8664706.558999473
2016-07-29 22:24:10 -0400 timed[62]: type:21;rtc_s:8662219.147125;mach:73565583165
2016-07-29 22:24:10 -0400 timed[62]: t_s:491538250.6569139;adjust:0.0002909302711486816;didSet:1;mach:73565143972;rtc_s:8662219.128825292;unc_s:0.0002507879726646324;after_mach:73565143993;type:3;settimeofday:0
2016-07-29 22:24:11 -0400 timed[62]: type:21;rtc_s:8662220.203674;mach:73590940894
2016-07-29 22:24:11 -0400 timed[62]: unc_ns:500944;src:13;before_mach:73590286211;after_mach:73590286308;rtc_s:8662220.176395541;type:11;t_ns:491538251704414395
2016-07-29 22:24:11 -0400 timed[62]: t_s:491538251.7044726;rateSf:0.9999889588421562;sf_var:3.81927462612792e-11;rtc_s:8662220.176395541;utc_var:6.310505604059744e-08;type:29;utc_sf_cov:1.215776576036138e-10
2016-07-29 22:24:11 -0400 timed[62]: _utc_sf_cov:9.714794177498255e-11;_rateSf:0.9999889362982626;_rtc:8662220.176395541;_utc_var:5.042477730510861e-08;type:32;_utc:491538251.7044609;_sf_var:3.814568018111509e-11
2016-07-29 22:24:11 -0400 timed[62]: need_rtc_s:9162208.948664548;bb_want_rtc_s:inf;type:20;rtc_s:8662220.22084;want_rtc_s:8664708.948664548
2016-07-29 22:24:11 -0400 timed[62]: type:21;rtc_s:8662220.223541001;mach:73591416855
2016-07-29 22:24:11 -0400 timed[62]: t_s:491538251.7348488;adjust:-1.120567321777344e-05;didSet:1;mach:73591014681;rtc_s:8662220.206783751;unc_s:0.000224572280375521;after_mach:73591014701;type:3;settimeofday:0
2016-07-29 22:24:12 -0400 timed[62]: type:21;rtc_s:8662221.238311;mach:73615775390
2016-07-29 22:24:12 -0400 timed[62]: unc_ns:500959;src:13;before_mach:73615102985;after_mach:73615103109;rtc_s:8662221.210294126;type:11;t_ns:491538252738432536
2016-07-29 22:24:12 -0400 timed[62]: t_s:491538252.7383481;rateSf:0.9999889362982626;sf_var:3.817442256176543e-11;rtc_s:8662221.210294126;utc_var:5.066644546849425e-08;type:29;utc_sf_cov:1.366015648740661e-10
2016-07-29 22:24:12 -0400 timed[62]: _utc_sf_cov:1.136556000044059e-10;_rateSf:0.9999889745486503;_rtc:8662221.210294126;_utc_var:4.21556317097908e-08;type:32;_utc:491538252.7383623;_sf_var:3.811255805400773e-11
2016-07-29 22:24:12 -0400 timed[62]: type:21;rtc_s:8662221.241699001;mach:73615856259
2016-07-29 22:24:12 -0400 timed[62]: need_rtc_s:9162210.944375791;bb_want_rtc_s:inf;type:20;rtc_s:8662221.256484;want_rtc_s:8664710.944375791
2016-07-29 22:24:12 -0400 timed[62]: t_s:491538252.7681848;adjust:-0.0001481771469116211;didSet:1;mach:73615818293;rtc_s:8662221.240117084;unc_s:0.0002053371363601146;after_mach:73615818319;type:3;settimeofday:0
2016-07-29 22:29:01 -0400 timed[62]: type:21;rtc_s:8662509.560974;mach:76746934310
2016-07-29 22:29:01 -0400 timed[62]: type:22;mach:73646552290
2016-07-29 22:29:01 -0400 timed[62]: unc_ns:500904;src:13;before_mach:76746327788;after_mach:76746327917;rtc_s:8662509.535702251;type:11;t_ns:491538541060177206
2016-07-29 22:29:01 -0400 timed[62]: t_s:491538541.0605915;rateSf:0.9999889745486503;sf_var:4.612800439989097e-11;rtc_s:8662509.535702251;utc_var:3.498163043019137e-06;type:29;utc_sf_cov:1.225800287511683e-08
2016-07-29 22:29:01 -0400 timed[62]: _utc_sf_cov:8.203631062021992e-10;_rateSf:0.9999876201039583;_rtc:8662509.535702251;_utc_var:2.341134954208897e-07;type:32;_utc:491538541.0602049;_sf_var:6.049077418002111e-12
2016-07-29 22:29:01 -0400 timed[62]: type:21;rtc_s:8662509.563995;mach:76747007618
2016-07-29 22:29:01 -0400 timed[62]: need_rtc_s:9162485.343064144;bb_want_rtc_s:inf;type:20;rtc_s:8662509.573959;want_rtc_s:8664985.343064144
2016-07-29 22:29:01 -0400 timed[62]: t_s:491538541.0881526;adjust:-0.00339740514755249;didSet:1;mach:76746999344;rtc_s:8662509.56365025;unc_s:0.0004839036140928802;after_mach:76746999358;type:3;settimeofday:0
2016-07-29 22:29:02 -0400 timed[62]: type:21;rtc_s:8662510.629760001;mach:76772585090
2016-07-29 22:29:02 -0400 timed[62]: unc_ns:500883;src:13;before_mach:76772003533;after_mach:76772003609;rtc_s:8662510.605528459;type:11;t_ns:491538542129983941
2016-07-29 22:29:02 -0400 timed[62]: t_s:491538542.1300179;rateSf:0.9999876201039583;sf_var:6.078818586586041e-12;rtc_s:8662510.605528459;utc_var:2.358757220088747e-07;type:29;utc_sf_cov:8.268504766993363e-10
2016-07-29 22:29:02 -0400 timed[62]: _utc_sf_cov:4.261727307136404e-10;_rateSf:0.9999875624930994;_rtc:8662510.605528459;_utc_var:1.215743394849933e-07;type:32;_utc:491538542.1300014;_sf_var:4.674262905952562e-12
2016-07-29 22:29:02 -0400 timed[62]: type:21;rtc_s:8662510.63208;mach:76772640797
2016-07-29 22:29:02 -0400 timed[62]: need_rtc_s:9162493.171772446;bb_want_rtc_s:inf;type:20;rtc_s:8662510.640791999;want_rtc_s:8664993.171772446
2016-07-29 22:29:02 -0400 timed[62]: t_s:491538542.1559186;adjust:6.616115570068359e-06;didSet:1;mach:76772625580;rtc_s:8662510.631445957;unc_s:0.0003487095232436957;after_mach:76772625594;type:3;settimeofday:0
2016-07-29 22:29:03 -0400 timed[62]: type:21;rtc_s:8662511.733337;mach:76799073164
2016-07-29 22:29:03 -0400 timed[62]: unc_ns:500962;src:13;before_mach:76798396636;after_mach:76798396808;rtc_s:8662511.705148334;type:11;t_ns:491538543229687346
2016-07-29 22:29:03 -0400 timed[62]: t_s:491538543.2296076;rateSf:0.9999875624930994;sf_var:4.704832338470076e-12;rtc_s:8662511.705148334;utc_var:1.22517259765486e-07;type:29;utc_sf_cov:4.313294504825201e-10
2016-07-29 22:29:03 -0400 timed[62]: _utc_sf_cov:2.898354842978959e-10;_rateSf:0.9999876545279172;_rtc:8662511.705148334;_utc_var:8.232651231965858e-08;type:32;_utc:491538543.2296338;_sf_var:4.206694241047504e-12
2016-07-29 22:29:03 -0400 timed[62]: type:21;rtc_s:8662511.736297;mach:76799144276
2016-07-29 22:29:03 -0400 timed[62]: need_rtc_s:9162497.358849829;bb_want_rtc_s:inf;type:20;rtc_s:8662511.749128999;want_rtc_s:8664997.358849829
2016-07-29 22:29:03 -0400 timed[62]: t_s:491538543.2594111;adjust:-8.189678192138672e-05;didSet:1;mach:76799111373;rtc_s:8662511.734926041;unc_s:0.000286958125328245;after_mach:76799111393;type:3;settimeofday:0
2016-07-29 22:29:04 -0400 timed[62]: type:21;rtc_s:8662512.775726;mach:76824089670
2016-07-29 22:29:04 -0400 timed[62]: unc_ns:500944;src:13;before_mach:76823434641;after_mach:76823434819;rtc_s:8662512.748433124;type:11;t_ns:491538544272924504
2016-07-29 22:29:04 -0400 timed[62]: t_s:491538544.2729057;rateSf:0.9999876545279172;sf_var:4.235697558225917e-12;rtc_s:8662512.748433124;utc_var:8.293586349511086e-08;type:29;utc_sf_cov:2.942393937780895e-10
2016-07-29 22:29:04 -0400 timed[62]: _utc_sf_cov:2.211506186187688e-10;_rateSf:0.9999876710741439;_rtc:8662512.748433124;_utc_var:6.233467681577081e-08;type:32;_utc:491538544.2729104;_sf_var:3.976393609504995e-12
2016-07-29 22:29:04 -0400 timed[62]: type:21;rtc_s:8662512.779663;mach:76824185533
2016-07-29 22:29:04 -0400 timed[62]: need_rtc_s:9162500.26497639;bb_want_rtc_s:inf;type:20;rtc_s:8662512.791671;want_rtc_s:8665000.26497639
2016-07-29 22:29:04 -0400 timed[62]: t_s:491538544.3020097;adjust:-2.831220626831055e-05;didSet:1;mach:76824134408;rtc_s:8662512.777532792;unc_s:0.0002496988302207028;after_mach:76824134431;type:3;settimeofday:0
2016-07-29 22:29:12 -0400 timed[62]: type:21;rtc_s:8662521.224822;mach:77026866697
2016-07-29 22:29:12 -0400 timed[62]: type:9;src:10;rtc_s:8662521.222762
2016-07-29 22:36:03 -0400 timed[62]: type:21;rtc_s:8662931.815551;mach:86881046414
2016-07-29 22:36:03 -0400 timed[62]: t_s:491538963.333767;adjust:-0.005150020122528076;didSet:1;mach:86881020143;rtc_s:8662931.814456375;unc_s:0.001275930150606349;after_mach:86881020176;type:3;settimeofday:0
2016-07-29 22:38:19 -0400 timed[62]: type:21;rtc_s:8663067.552398;mach:90138728261
2016-07-29 22:38:19 -0400 timed[62]: unc_ns:500925;src:13;before_mach:90138096870;after_mach:90138096997;rtc_s:8663067.526090041;type:11;t_ns:491539099043989172
2016-07-29 22:38:19 -0400 timed[62]: t_s:491539099.0437275;rateSf:0.9999876710741439;sf_var:1.93992124717845e-11;rtc_s:8663067.526090041;utc_var:3.113830374500947e-06;type:29;utc_sf_cov:6.705282604009543e-09
2016-07-29 22:38:19 -0400 timed[62]: _utc_sf_cov:5.000450698474102e-10;_rateSf:0.9999881926374159;_rtc:8663067.526090041;_utc_var:2.322132591666225e-07;type:32;_utc:491539099.0439697;_sf_var:6.036933128538992e-12
2016-07-29 22:38:19 -0400 timed[62]: type:21;rtc_s:8663067.567840001;mach:90139097772
2016-07-29 22:38:19 -0400 timed[62]: need_rtc_s:9163043.431834545;bb_want_rtc_s:inf;type:20;rtc_s:8663067.569473;want_rtc_s:8665543.431834545
2016-07-29 22:38:19 -0400 timed[62]: t_s:491539099.0771699;adjust:-0.001281082630157471;didSet:1;mach:90138892587;rtc_s:8663067.559290625;unc_s:0.0004819238267002765;after_mach:90138892593;type:3;settimeofday:0
2016-07-29 22:38:20 -0400 timed[62]: type:21;rtc_s:8663068.681549;mach:90165830298
2016-07-29 22:38:20 -0400 timed[62]: unc_ns:501059;src:13;before_mach:90165038141;after_mach:90165038227;rtc_s:8663068.648542458;type:11;t_ns:491539100166530730
2016-07-29 22:38:20 -0400 timed[62]: t_s:491539100.1664088;rateSf:0.9999881926374159;sf_var:6.068137305742394e-12;rtc_s:8663068.648542458;utc_var:2.333434317948412e-07;type:29;utc_sf_cov:5.068387526332191e-10
2016-07-29 22:38:20 -0400 timed[62]: _utc_sf_cov:2.626880623760051e-10;_rateSf:0.999988320174297;_rtc:8663068.648542458;_utc_var:1.209389251470919e-07;type:32;_utc:491539100.1664675;_sf_var:5.537824408770414e-12
2016-07-29 22:38:20 -0400 timed[62]: type:21;rtc_s:8663068.689177999;mach:90166012454
2016-07-29 22:38:20 -0400 timed[62]: need_rtc_s:9163051.260405328;bb_want_rtc_s:inf;type:20;rtc_s:8663068.705855999;want_rtc_s:8665551.260405328
2016-07-29 22:38:20 -0400 timed[62]: t_s:491539100.2052417;adjust:-6.234645843505859e-05;didSet:1;mach:90165967792;rtc_s:8663068.687317083;unc_s:0.0003477941223053291;after_mach:90165967816;type:3;settimeofday:0
2016-07-29 22:38:21 -0400 timed[62]: type:21;rtc_s:8663069.798461;mach:90192634398
2016-07-29 22:38:21 -0400 timed[62]: unc_ns:500933;src:13;before_mach:90191992604;after_mach:90191992749;rtc_s:8663069.771719582;type:11;t_ns:491539101289625971
2016-07-29 22:38:21 -0400 timed[62]: t_s:491539101.2896315;rateSf:0.999988320174297;sf_var:5.569048732822822e-12;rtc_s:8663069.771719582;utc_var:1.215360148363103e-07;type:29;utc_sf_cov:2.689255552929493e-10
2016-07-29 22:38:21 -0400 timed[62]: _utc_sf_cov:1.811759982305686e-10;_rateSf:0.9999883161290234;_rtc:8663069.771719582;_utc_var:8.187919807378389e-08;type:32;_utc:491539101.2896297;_sf_var:5.374883255276977e-12
2016-07-29 22:38:21 -0400 timed[62]: need_rtc_s:9163055.464448871;bb_want_rtc_s:inf;type:20;rtc_s:8663069.820784999;want_rtc_s:8665555.464448871
2016-07-29 22:38:21 -0400 timed[62]: type:21;rtc_s:8663069.826019;mach:90193295274
2016-07-29 22:38:21 -0400 timed[62]: t_s:491539101.3194103;adjust:4.225969314575195e-05;didSet:1;mach:90192706830;rtc_s:8663069.801500501;unc_s:0.0002861678704920018;after_mach:90192706854;type:3;settimeofday:0
2016-07-29 22:38:22 -0400 timed[62]: type:21;rtc_s:8663070.847717;mach:90217817815
2016-07-29 22:38:22 -0400 timed[62]: unc_ns:500916;src:13;before_mach:90217196333;after_mach:90217196506;rtc_s:8663070.821821917;type:11;t_ns:491539102339772171
2016-07-29 22:38:22 -0400 timed[62]: t_s:491539102.3397198;rateSf:0.9999883161290234;sf_var:5.404076100175231e-12;rtc_s:8663070.821821917;utc_var:8.226564244554971e-08;type:29;utc_sf_cov:1.868355034217251e-10
2016-07-29 22:38:22 -0400 timed[62]: _utc_sf_cov:1.407043233543613e-10;_rateSf:0.9999883455085673;_rtc:8663070.821821917;_utc_var:6.19535973817862e-08;type:32;_utc:491539102.3397327;_sf_var:5.299306454000901e-12
2016-07-29 22:38:22 -0400 timed[62]: type:21;rtc_s:8663070.856445;mach:90218026062
2016-07-29 22:38:22 -0400 timed[62]: need_rtc_s:9163058.376582125;bb_want_rtc_s:inf;type:20;rtc_s:8663070.862501999;want_rtc_s:8665558.376582125
2016-07-29 22:38:22 -0400 timed[62]: t_s:491539102.3673068;adjust:-2.318620681762695e-05;didSet:1;mach:90217856894;rtc_s:8663070.849396333;unc_s:0.0002489236819153953;after_mach:90217856908;type:3;settimeofday:0
2016-07-29 22:38:23 -0400 timed[62]: type:21;rtc_s:8663071.933227001;mach:90243868562
2016-07-29 22:38:23 -0400 timed[62]: unc_ns:500527;src:13;before_mach:90243714253;after_mach:90243714376;rtc_s:8663071.926797459;type:11;t_ns:491539103444673140
2016-07-29 22:38:23 -0400 timed[62]: t_s:491539103.4446954;rateSf:0.9999883455085673;sf_var:5.330024774069991e-12;rtc_s:8663071.926797459;utc_var:6.227102985500827e-08;type:29;utc_sf_cov:1.465768988720636e-10
2016-07-29 22:38:23 -0400 timed[62]: _utc_sf_cov:1.173968267115888e-10;_rateSf:0.9999883350904498;_rtc:8663071.926797459;_utc_var:4.987430732465773e-08;type:32;_utc:491539103.4446909;_sf_var:5.261339150978414e-12
2016-07-29 22:38:23 -0400 timed[62]: type:21;rtc_s:8663071.943542;mach:90244116633
2016-07-29 22:38:23 -0400 timed[62]: need_rtc_s:9163060.760519283;bb_want_rtc_s:inf;type:20;rtc_s:8663071.946456;want_rtc_s:8665560.760519283
2016-07-29 22:38:23 -0400 timed[62]: t_s:491539103.4531245;adjust:-2.747774124145508e-05;didSet:1;mach:90243917172;rtc_s:8663071.935231125;unc_s:0.0002233324733544162;after_mach:90243917187;type:3;settimeofday:0
2016-07-29 22:38:24 -0400 timed[62]: type:21;rtc_s:8663073.048736;mach:90270641170
2016-07-29 22:38:24 -0400 timed[62]: unc_ns:500933;src:13;before_mach:90269999776;after_mach:90269999937;rtc_s:8663073.02201125;type:11;t_ns:491539104539895460
2016-07-29 22:38:24 -0400 timed[62]: t_s:491539104.539892;rateSf:0.9999883350904498;sf_var:5.291786094378097e-12;rtc_s:8663073.02201125;utc_var:5.0137779686435e-08;type:29;utc_sf_cov:1.231757908668969e-10
2016-07-29 22:38:24 -0400 timed[62]: _utc_sf_cov:1.026632301893344e-10;_rateSf:0.9999883365292049;_rtc:8663073.02201125;_utc_var:4.178829606779439e-08;type:32;_utc:491539104.5398926;_sf_var:5.241391942503685e-12
2016-07-29 22:38:24 -0400 timed[62]: type:21;rtc_s:8663073.070983;mach:90271180370
2016-07-29 22:38:24 -0400 timed[62]: need_rtc_s:9163062.800918344;bb_want_rtc_s:inf;type:20;rtc_s:8663073.081831999;want_rtc_s:8665562.800918346
2016-07-29 22:38:24 -0400 timed[62]: t_s:491539104.5694181;adjust:-0.0002278685569763184;didSet:1;mach:90270713670;rtc_s:8663073.051537167;unc_s:0.0002044386499942829;after_mach:90270713690;type:3;settimeofday:0
2016-07-29 22:43:39 -0400 timed[62]: type:21;rtc_s:8663387.989837;mach:97829225525
2016-07-29 22:43:39 -0400 timed[62]: type:6;src:10;rtc_s:8663387.988754001
2016-07-29 22:57:57 -0400 timed[62]: type:21;rtc_s:8664246.411071001;mach:102981271928
2016-07-29 22:57:57 -0400 timed[62]: type:22;mach:102972164843
2016-07-29 22:57:57 -0400 timed[62]: type:9;src:10;rtc_s:8664246.409164
2016-07-29 23:19:55 -0400 timed[62]: type:21;rtc_s:8665564.346252;mach:104747219930
2016-07-29 23:19:55 -0400 timed[62]: type:22;mach:104739578058
2016-07-29 23:19:55 -0400 timed[62]: type:9;src:10;rtc_s:8665564.334731
2016-07-29 23:19:57 -0400 timed[62]: port:123;ipv4:301798525;delay:0.03430469403974712;mach:104782153862;type:2;tv_usec:290458;dispersion:0.00103759765625;attempts:0;num:0;tv_sec:1469848797;result:0;slept:0;more:1
2016-07-29 23:19:57 -0400 timed[62]: type:21;rtc_s:8665565.803466;mach:104782195438
2016-07-29 23:19:57 -0400 timed[62]: port:123;ipv4:301799677;delay:0.05357015854679048;mach:104783454898;type:2;tv_usec:345265;dispersion:0.00103759765625;attempts:0;num:1;tv_sec:1469848797;result:0;slept:0;more:1
2016-07-29 23:19:57 -0400 timed[62]: type:21;rtc_s:8665565.857757;mach:104783496511
2016-07-29 23:19:57 -0400 timed[62]: port:123;ipv4:301798653;delay:0.03546417132019997;mach:104784322689;type:2;tv_usec:380573;dispersion:0.0010223388671875;attempts:0;num:2;tv_sec:1469848797;result:0;slept:0;more:0
2016-07-29 23:19:57 -0400 timed[62]: type:21;rtc_s:8665565.893432001;mach:104784353094
2016-07-29 23:19:57 -0400 timed[62]: mach:104782153862;unc_s:0.01870027962528605;t_s:491541597.2909684;type:5;src:5;rtc_s:8665565.801813291
2016-07-29 23:19:57 -0400 timed[62]: t_s:491541597.2906201;rateSf:0.9999883365292049;sf_var:7.454067043924705e-11;rtc_s:8665565.801813291;utc_var:0.0001766641522991345;type:29;utc_sf_cov:9.95422200653961e-08
2016-07-29 23:19:57 -0400 timed[62]: _utc_sf_cov:6.613278945474258e-08;_rateSf:0.9999884023801932;_rtc:8665565.801813291;_utc_var:0.0001173702292406547;type:32;_utc:491541597.290737;_sf_var:5.571597481863163e-11
2016-07-29 23:19:57 -0400 timed[62]: type:21;rtc_s:8665565.897094;mach:104784439892
2016-07-29 23:19:57 -0400 timed[62]: need_rtc_s:9165024.114105197;bb_want_rtc_s:inf;type:20;rtc_s:8665565.910201;want_rtc_s:8667524.114105197
2016-07-29 23:19:57 -0400 timed[62]: t_s:491541597.3851546;adjust:-0.02876538038253784;didSet:1;mach:104784419203;rtc_s:8665565.896231959;unc_s:0.01083433403310672;after_mach:104784419225;type:3;settimeofday:0
2016-07-29 23:43:14 -0400 timed[62]: type:21;rtc_s:8666962.737365;mach:106172257900
2016-07-29 23:43:14 -0400 timed[62]: type:22;mach:106162510743
2016-07-29 23:43:14 -0400 timed[62]: type:9;src:10;rtc_s:8666962.735152001
2016-07-29 23:55:37 -0400 timed[62]: type:22;mach:106874247931
2016-07-29 23:55:37 -0400 timed[62]: port:123;ipv4:301799549;delay:0.05858104163780808;mach:106904430799;type:2;tv_usec:505573;dispersion:0.0010986328125;attempts:0;num:0;tv_sec:1469850937;result:0;slept:0;more:1
2016-07-29 23:55:37 -0400 timed[62]: type:21;rtc_s:8667706.043914;mach:106904474113
2016-07-29 23:55:37 -0400 timed[62]: port:123;ipv4:301798525;delay:0.03621940384618938;mach:106905317556;type:2;tv_usec:542674;dispersion:0.0010986328125;attempts:0;num:1;tv_sec:1469850937;result:0;slept:0;more:1
2016-07-29 23:55:37 -0400 timed[62]: type:21;rtc_s:8667706.082916001;mach:106905409334
2016-07-29 23:55:37 -0400 timed[62]: port:123;ipv4:301798653;delay:0.03390550846233964;mach:106906149729;type:2;tv_usec:578813;dispersion:0.0010986328125;attempts:0;num:2;tv_sec:1469850937;result:0;slept:0;more:0
2016-07-29 23:55:37 -0400 timed[62]: type:21;rtc_s:8667706.116424;mach:106906213824
2016-07-29 23:55:37 -0400 timed[62]: mach:106904430799;unc_s:0.01841108529378316;t_s:491543737.5075507;type:5;src:5;rtc_s:8667706.04214371
2016-07-29 23:55:37 -0400 timed[62]: t_s:491543737.5062457;rateSf:0.9999884023801932;sf_var:1.152146560042679e-10;rtc_s:8667706.04214371;utc_var:0.0007465116689494907;type:29;utc_sf_cov:2.490491043502713e-07
2016-07-29 23:55:37 -0400 timed[62]: _utc_sf_cov:7.777178125505574e-08;_rateSf:0.9999887017780223;_rtc:8667706.04214371;_utc_var:0.0002331168480743951;type:32;_utc:491543737.5071431;_sf_var:5.807360156850447e-11
2016-07-29 23:55:37 -0400 timed[62]: type:21;rtc_s:8667706.120604999;mach:106906313622
2016-07-29 23:55:37 -0400 timed[62]: need_rtc_s:9166942.633917276;bb_want_rtc_s:inf;type:20;rtc_s:8667706.133772999;want_rtc_s:8669442.633917276
2016-07-29 23:55:37 -0400 timed[62]: t_s:491543737.585233;adjust:-0.02368295192718506;didSet:1;mach:106906304729;rtc_s:8667706.120234458;unc_s:0.01526856565341681;after_mach:106906304751;type:3;settimeofday:0
2016-07-29 23:56:18 -0400 timed[62]: type:21;rtc_s:8667747.303863;mach:107462823717
2016-07-29 23:56:18 -0400 timed[62]: type:22;mach:107081560224
2016-07-29 23:56:18 -0400 timed[62]: unc_ns:500888;src:13;before_mach:107462235858;after_mach:107462235960;rtc_s:8667747.279368876;type:11;t_ns:491543778743933670
2016-07-29 23:56:18 -0400 timed[62]: t_s:491543778.7439024;rateSf:0.9999887017780223;sf_var:5.921999642810788e-11;rtc_s:8667747.279368876;utc_var:0.0002396304374698351;type:29;utc_sf_cov:8.019021251059002e-08
2016-07-29 23:56:18 -0400 timed[62]: _utc_sf_cov:8.387020533503005e-11;_rateSf:0.9999887122388086;_rtc:8667747.279368876;_utc_var:2.506272694122615e-07;type:32;_utc:491543778.7439336;_sf_var:3.241311546285866e-11
2016-07-29 23:56:18 -0400 timed[62]: need_rtc_s:9167722.248025054;bb_want_rtc_s:inf;type:20;rtc_s:8667747.314895;want_rtc_s:8670222.248025054
2016-07-29 23:56:18 -0400 timed[62]: type:21;rtc_s:8667747.316009;mach:107463114462
2016-07-29 23:56:18 -0400 timed[62]: t_s:491543778.7704449;adjust:-0.0001660585403442383;didSet:1;mach:107462871377;rtc_s:8667747.305880459;unc_s:0.0005006347408167011;after_mach:107462871383;type:3;settimeofday:0
2016-07-29 23:56:19 -0400 timed[62]: type:21;rtc_s:8667748.331176;mach:107487479993
2016-07-29 23:56:19 -0400 timed[62]: unc_ns:500935;src:13;before_mach:107486836248;after_mach:107486836413;rtc_s:8667748.304353291;type:11;t_ns:491543779768938308
2016-07-29 23:56:19 -0400 timed[62]: t_s:491543779.7689065;rateSf:0.9999887122388086;sf_var:3.244161002961291e-11;rtc_s:8667748.304353291;utc_var:2.508332636917069e-07;type:29;utc_sf_cov:1.171077467895967e-10
2016-07-29 23:56:19 -0400 timed[62]: _utc_sf_cov:5.856590060254329e-11;_rateSf:0.9999887196673346;_rtc:8667748.304353291;_utc_var:1.254423929406953e-07;type:32;_utc:491543779.7689224;_sf_var:3.241427831292203e-11
2016-07-29 23:56:19 -0400 timed[62]: need_rtc_s:9167730.595429486;bb_want_rtc_s:inf;type:20;rtc_s:8667748.349319;want_rtc_s:8670230.595429486
2016-07-29 23:56:19 -0400 timed[62]: type:21;rtc_s:8667748.351988999;mach:107487979591
2016-07-29 23:56:19 -0400 timed[62]: t_s:491543779.7987152;adjust:-6.282329559326172e-05;didSet:1;mach:107487551370;rtc_s:8667748.334146457;unc_s:0.0003541875772901101;after_mach:107487551389;type:3;settimeofday:0
2016-07-29 23:56:20 -0400 timed[62]: type:21;rtc_s:8667749.364227001;mach:107512273779
2016-07-29 23:56:20 -0400 timed[62]: unc_ns:500932;src:13;before_mach:107511633676;after_mach:107511633855;rtc_s:8667749.337556042;type:11;t_ns:491543780802149968
2016-07-29 23:56:20 -0400 timed[62]: t_s:491543780.8021135;rateSf:0.9999887196673346;sf_var:3.244300134938894e-11;rtc_s:8667749.337556042;utc_var:1.255980265595192e-07;type:29;utc_sf_cov:9.207126047457983e-11
2016-07-29 23:56:20 -0400 timed[62]: _utc_sf_cov:6.135942282991992e-11;_rateSf:0.9999887285871222;_rtc:8667749.337556042;_utc_var:8.370280127094374e-08;type:32;_utc:491543780.8021256;_sf_var:3.24204876389359e-11
2016-07-29 23:56:20 -0400 timed[62]: need_rtc_s:9167734.871837838;bb_want_rtc_s:inf;type:20;rtc_s:8667749.380263999;want_rtc_s:8670234.871837838
2016-07-29 23:56:20 -0400 timed[62]: type:21;rtc_s:8667749.382995;mach:107512723652
2016-07-29 23:56:20 -0400 timed[62]: t_s:491543780.8319253;adjust:3.278255462646484e-06;didSet:1;mach:107512348316;rtc_s:8667749.367356;unc_s:0.0002893249876390155;after_mach:107512348337;type:3;settimeofday:0
2016-07-29 23:56:21 -0400 timed[62]: type:21;rtc_s:8667750.481262;mach:107539084810
2016-07-29 23:56:21 -0400 timed[62]: unc_ns:500941;src:13;before_mach:107538433648;after_mach:107538433814;rtc_s:8667750.454130251;type:11;t_ns:491543781918798747
2016-07-29 23:56:21 -0400 timed[62]: t_s:491543781.9186873;rateSf:0.9999887285871222;sf_var:3.24515284019512e-11;rtc_s:8667750.454130251;utc_var:8.388025872165826e-08;type:29;utc_sf_cov:9.757663283441486e-11
2016-07-29 23:56:21 -0400 timed[62]: _utc_sf_cov:7.313160195089303e-11;_rateSf:0.9999887610698535;_rtc:8667750.454130251;_utc_var:6.286646212500474e-08;type:32;_utc:491543781.9187152;_sf_var:3.242309186826021e-11
2016-07-29 23:56:21 -0400 timed[62]: type:21;rtc_s:8667750.488005999;mach:107539245361
2016-07-29 23:56:21 -0400 timed[62]: need_rtc_s:9167737.9175376;bb_want_rtc_s:inf;type:20;rtc_s:8667750.505996;want_rtc_s:8670237.9175376
2016-07-29 23:56:22 -0400 timed[62]: t_s:491543781.9498459;adjust:-4.51207160949707e-05;didSet:1;mach:107539179488;rtc_s:8667750.485261291;unc_s:0.0002507437402222843;after_mach:107539179510;type:3;settimeofday:0
2016-07-30 00:05:09 -0400 timed[62]: type:21;rtc_s:8668277.693908;mach:120192192810
2016-07-30 00:05:09 -0400 timed[62]: type:9;src:10;rtc_s:8668277.691664999
2016-07-30 00:27:28 -0400 timed[62]: type:21;rtc_s:8669616.926452;mach:136897964376
2016-07-30 00:27:28 -0400 timed[62]: type:22;mach:136889591443
2016-07-30 00:27:28 -0400 timed[62]: type:9;src:10;rtc_s:8669616.914626
2016-07-30 00:27:37 -0400 timed[62]: type:21;rtc_s:8669626.125151999;mach:137118734972
2016-07-30 00:27:37 -0400 timed[62]: unc_ns:633226;src:16;before_mach:137118030492;after_mach:137118030672;rtc_s:8669626.095798666;type:11;t_ns:491545657537837859
2016-07-30 00:27:37 -0400 timed[62]: t_s:491545657.5393034;rateSf:0.9999887610698535;sf_var:8.456593025018879e-11;rtc_s:8669626.095798666;utc_var:0.0001755493866992245;type:29;utc_sf_cov:1.097878739181767e-07
2016-07-30 00:27:37 -0400 timed[62]: _utc_sf_cov:2.50196816128131e-10;_rateSf:0.9999878466411379;_rtc:8669626.095798666;_utc_var:4.000614645122502e-07;type:32;_utc:491545657.5378412;_sf_var:1.606151158437991e-11
2016-07-30 00:27:37 -0400 timed[62]: need_rtc_s:9169594.470592558;bb_want_rtc_s:inf;type:20;rtc_s:8669626.157546001;want_rtc_s:8672094.470592558
2016-07-30 00:27:37 -0400 timed[62]: type:21;rtc_s:8669626.162384;mach:137119627326
2016-07-30 00:27:37 -0400 timed[62]: t_s:491545657.5848737;adjust:-0.02228432893753052;didSet:1;mach:137119158071;rtc_s:8669626.142831707;unc_s:0.0006325262382384496;after_mach:137119158099;type:3;settimeofday:0
2016-07-30 00:27:38 -0400 timed[62]: type:21;rtc_s:8669627.2052;mach:137144655056
2016-07-30 00:27:38 -0400 timed[62]: unc_ns:500937;src:13;before_mach:137144009453;after_mach:137144009623;rtc_s:8669627.178299874;type:11;t_ns:491545658620279490
2016-07-30 00:27:38 -0400 timed[62]: t_s:491545658.6203293;rateSf:0.9999878466411379;sf_var:1.60916051179737e-11;rtc_s:8669627.178299874;utc_var:4.006219740001505e-07;type:29;utc_sf_cov:2.675997099703484e-10
2016-07-30 00:27:38 -0400 timed[62]: _utc_sf_cov:1.030617579862398e-10;_rateSf:0.9999878262003374;_rtc:8669627.178299874;_utc_var:1.542931602315571e-07;type:32;_utc:491545658.6202987;_sf_var:1.598170024233639e-11
2016-07-30 00:27:38 -0400 timed[62]: need_rtc_s:9169607.538215848;bb_want_rtc_s:inf;type:20;rtc_s:8669627.22264;want_rtc_s:8672107.538215848
2016-07-30 00:27:38 -0400 timed[62]: type:21;rtc_s:8669627.225920999;mach:137145152558
2016-07-30 00:27:38 -0400 timed[62]: t_s:491545658.6503223;adjust:-0.01838475465774536;didSet:1;mach:137144730226;rtc_s:8669627.208323833;unc_s:0.000392813717704577;after_mach:137144730315;type:3;settimeofday:0
2016-07-30 00:27:39 -0400 timed[62]: type:21;rtc_s:8669628.283782;mach:137170541741
2016-07-30 00:27:39 -0400 timed[62]: unc_ns:500936;src:13;before_mach:137169897319;after_mach:137169897498;rtc_s:8669628.256931083;type:11;t_ns:491545659698924287
2016-07-30 00:27:39 -0400 timed[62]: t_s:491545659.6989167;rateSf:0.9999878262003374;sf_var:1.601168618995241e-11;rtc_s:8669628.256931083;utc_var:1.54534096951641e-07;type:29;utc_sf_cov:1.20316290533324e-10
2016-07-30 00:27:39 -0400 timed[62]: _utc_sf_cov:7.446105131820821e-11;_rateSf:0.9999878284465377;_rtc:8669628.256931083;_utc_var:9.563768357986465e-08;type:32;_utc:491545659.6989196;_sf_var:1.597598447497356e-11
2016-07-30 00:27:39 -0400 timed[62]: type:21;rtc_s:8669628.296295;mach:137170848537
2016-07-30 00:27:39 -0400 timed[62]: need_rtc_s:9169612.794259615;bb_want_rtc_s:inf;type:20;rtc_s:8669628.30397;want_rtc_s:8672112.794259615
2016-07-30 00:27:39 -0400 timed[62]: t_s:491545659.729378;adjust:-0.01440697908401489;didSet:1;mach:137170634814;rtc_s:8669628.287389874;unc_s:0.000309261579000016;after_mach:137170634898;type:3;settimeofday:0
2016-07-30 00:27:40 -0400 timed[62]: type:21;rtc_s:8669629.320373001;mach:137195419628
2016-07-30 00:27:40 -0400 timed[62]: unc_ns:500978;src:13;before_mach:137194724240;after_mach:137194724419;rtc_s:8669629.291398501;type:11;t_ns:491545660733363776
2016-07-30 00:27:40 -0400 timed[62]: t_s:491545660.7333744;rateSf:0.9999878284465377;sf_var:1.600474266918693e-11;rtc_s:8669629.291398501;utc_var:9.580884516694343e-08;type:29;utc_sf_cov:9.100256143145027e-11
2016-07-30 00:27:40 -0400 timed[62]: _utc_sf_cov:6.5860852487441e-11;_rateSf:0.9999878256467679;_rtc:8669629.291398501;_utc_var:6.933928143643874e-08;type:32;_utc:491545660.7333715;_sf_var:1.598086220273644e-11
2016-07-30 00:27:40 -0400 timed[62]: need_rtc_s:9169616.125222003;bb_want_rtc_s:inf;type:20;rtc_s:8669629.334701;want_rtc_s:8672116.125222003
2016-07-30 00:27:40 -0400 timed[62]: type:21;rtc_s:8669629.337646;mach:137195833655
2016-07-30 00:27:40 -0400 timed[62]: t_s:491545660.7653646;adjust:-0.009998440742492676;didSet:1;mach:137195491559;rtc_s:8669629.323392;unc_s:0.0002633360457232483;after_mach:137195491644;type:3;settimeofday:0
2016-07-30 00:27:41 -0400 timed[62]: type:21;rtc_s:8669630.374755001;mach:137220727823
2016-07-30 00:27:41 -0400 timed[62]: unc_ns:500937;src:13;before_mach:137220081411;after_mach:137220081583;rtc_s:8669630.347821167;type:11;t_ns:491545661789896224
2016-07-30 00:27:41 -0400 timed[62]: t_s:491545661.7897813;rateSf:0.9999878256467679;sf_var:1.601023075284332e-11;rtc_s:8669630.347821167;utc_var:6.949628125976827e-08;type:29;utc_sf_cov:8.275891033707945e-11
2016-07-30 00:27:41 -0400 timed[62]: _utc_sf_cov:6.481006132789817e-11;_rateSf:0.9999878553420442;_rtc:8669630.347821167;_utc_var:5.442384671525134e-08;type:32;_utc:491545661.7898062;_sf_var:1.598885655613987e-11
2016-07-30 00:27:41 -0400 timed[62]: type:21;rtc_s:8669630.382476;mach:137220910842
2016-07-30 00:27:41 -0400 timed[62]: need_rtc_s:9169618.683361609;bb_want_rtc_s:inf;type:20;rtc_s:8669630.395431001;want_rtc_s:8672118.683361609
2016-07-30 00:27:41 -0400 timed[62]: t_s:491545661.8216802;adjust:-0.005798757076263428;didSet:1;mach:137220844113;rtc_s:8669630.379695626;unc_s:0.0002333007159670165;after_mach:137220844202;type:3;settimeofday:0
2016-07-30 00:27:42 -0400 timed[62]: type:21;rtc_s:8669631.415007999;mach:137245690879
2016-07-30 00:27:42 -0400 timed[62]: unc_ns:501053;src:13;before_mach:137244904983;after_mach:137244905155;rtc_s:8669631.382262332;type:11;t_ns:491545662824195819
2016-07-30 00:27:42 -0400 timed[62]: t_s:491545662.8242348;rateSf:0.9999878553420442;sf_var:1.601761402054428e-11;rtc_s:8669631.382262332;utc_var:5.457505053565957e-08;type:29;utc_sf_cov:8.136446669335866e-11
2016-07-30 00:27:42 -0400 timed[62]: _utc_sf_cov:6.683556312464055e-11;_rateSf:0.9999878449644533;_rtc:8669631.382262332;_utc_var:4.482981801937201e-08;type:32;_utc:491545662.8242278;_sf_var:1.59959532711926e-11
2016-07-30 00:27:42 -0400 timed[62]: type:21;rtc_s:8669631.426085999;mach:137245958145
2016-07-30 00:27:42 -0400 timed[62]: need_rtc_s:9169620.795735752;bb_want_rtc_s:inf;type:20;rtc_s:8669631.436201999;want_rtc_s:8672120.795735752
2016-07-30 00:27:42 -0400 timed[62]: t_s:491545662.8599746;adjust:-0.001726448535919189;didSet:1;mach:137245764310;rtc_s:8669631.41800954;unc_s:0.0002117453776348055;after_mach:137245764397;type:3;settimeofday:0
2016-07-30 00:42:44 -0400 timed[62]: type:21;rtc_s:8670532.712859999;mach:139022300196
2016-07-30 00:42:44 -0400 timed[62]: type:22;mach:138938974369
2016-07-30 00:42:44 -0400 timed[62]: unc_ns:501544;src:14;before_mach:139021719019;after_mach:139021719145;rtc_s:8670532.688644292;type:11;t_ns:491546564120806053
2016-07-30 00:42:44 -0400 timed[62]: t_s:491546564.1196544;rateSf:0.9999878449644533;sf_var:4.105227068966581e-11;rtc_s:8670532.688644292;utc_var:1.994453202029459e-05;type:29;utc_sf_cov:2.577579973081201e-08
2016-07-30 00:42:44 -0400 timed[62]: _utc_sf_cov:3.210433781896685e-10;_rateSf:0.999989314828868;_rtc:8670532.688644292;_utc_var:2.484136284063872e-07;type:32;_utc:491546564.1207917;_sf_var:8.15519888701957e-12
2016-07-30 00:42:44 -0400 timed[62]: type:21;rtc_s:8670532.718901999;mach:139022444711
2016-07-30 00:42:44 -0400 timed[62]: need_rtc_s:9170507.768089101;bb_want_rtc_s:inf;type:20;rtc_s:8670532.725814;want_rtc_s:8673007.768089101
2016-07-30 00:42:44 -0400 timed[62]: t_s:491546564.1484981;adjust:-0.009579956531524658;didSet:1;mach:139022383486;rtc_s:8670532.716350958;unc_s:0.0004984318590722698;after_mach:139022383494;type:3;settimeofday:0
2016-07-30 00:42:45 -0400 timed[62]: type:21;rtc_s:8670533.746398;mach:139047105909
2016-07-30 00:42:45 -0400 timed[62]: unc_ns:500905;src:13;before_mach:139046497847;after_mach:139046497963;rtc_s:8670533.721062083;type:11;t_ns:491546565153240988
2016-07-30 00:42:45 -0400 timed[62]: t_s:491546565.1531984;rateSf:0.999989314828868;sf_var:8.183900101608376e-12;rtc_s:8670533.721062083;utc_var:2.490852329107866e-07;type:29;utc_sf_cov:3.294777664317611e-10
2016-07-30 00:42:45 -0400 timed[62]: _utc_sf_cov:1.653389728413932e-10;_rateSf:0.9999893428729721;_rtc:8670533.721062083;_utc_var:1.249962842878451e-07;type:32;_utc:491546565.1532196;_sf_var:7.966785331435196e-12
2016-07-30 00:42:45 -0400 timed[62]: type:21;rtc_s:8670533.750151999;mach:139047194932
2016-07-30 00:42:45 -0400 timed[62]: need_rtc_s:9170516.043655295;bb_want_rtc_s:inf;type:20;rtc_s:8670533.758680999;want_rtc_s:8673016.043655295
2016-07-30 00:42:45 -0400 timed[62]: t_s:491546565.1808705;adjust:-0.005580484867095947;didSet:1;mach:139047160402;rtc_s:8670533.748713249;unc_s:0.0003535642639108832;after_mach:139047160444;type:3;settimeofday:0
2016-07-30 00:42:46 -0400 timed[62]: type:21;rtc_s:8670534.831267999;mach:139073164454
2016-07-30 00:42:46 -0400 timed[62]: unc_ns:500953;src:13;before_mach:139072499110;after_mach:139072499253;rtc_s:8670534.803545333;type:11;t_ns:491546566236612066
2016-07-30 00:42:46 -0400 timed[62]: t_s:491546566.2356914;rateSf:0.9999893428729721;sf_var:7.996878365803204e-12;rtc_s:8670534.803545333;utc_var:1.253635846193952e-07;type:29;utc_sf_cov:1.739791721270105e-10
2016-07-30 00:42:46 -0400 timed[62]: _utc_sf_cov:1.16021126704716e-10;_rateSf:0.9999897685366369;_rtc:8670534.803545333;_utc_var:8.360095152462291e-08;type:32;_utc:491546566.2359981;_sf_var:7.916444380198181e-12
2016-07-30 00:42:46 -0400 timed[62]: type:21;rtc_s:8670534.839599;mach:139073345998
2016-07-30 00:42:46 -0400 timed[62]: need_rtc_s:9170520.346630765;bb_want_rtc_s:inf;type:20;rtc_s:8670534.851638;want_rtc_s:8673020.346630765
2016-07-30 00:42:46 -0400 timed[62]: t_s:491546566.2688605;adjust:-0.001148462295532227;didSet:1;mach:139073269416;rtc_s:8670534.836408084;unc_s:0.000289153845913852;after_mach:139073269558;type:3;settimeofday:0
2016-07-30 00:42:47 -0400 timed[62]: type:21;rtc_s:8670535.94815;mach:139099948406
2016-07-30 00:42:47 -0400 timed[62]: unc_ns:500934;src:13;before_mach:139099306038;after_mach:139099306198;rtc_s:8670535.921384666;type:11;t_ns:491546567353551474
2016-07-30 00:42:47 -0400 timed[62]: t_s:491546567.353826;rateSf:0.9999897685366369;sf_var:7.947520313646406e-12;rtc_s:8670535.921384666;utc_var:8.387024253615276e-08;type:29;utc_sf_cov:1.248878085581496e-10
2016-07-30 00:42:47 -0400 timed[62]: _utc_sf_cov:9.360285884948476e-11;_rateSf:0.9999896661291756;_rtc:8670535.921384666;_utc_var:6.286037495908381e-08;type:32;_utc:491546567.3537572;_sf_var:7.90093515195115e-12
2016-07-30 00:42:47 -0400 timed[62]: type:21;rtc_s:8670535.961242;mach:139100262482
2016-07-30 00:42:47 -0400 timed[62]: need_rtc_s:9170523.385398969;bb_want_rtc_s:inf;type:20;rtc_s:8670535.978286;want_rtc_s:8673023.385398969
2016-07-30 00:42:47 -0400 timed[62]: t_s:491546567.3891889;adjust:0.0002458691596984863;didSet:1;mach:139100156275;rtc_s:8670535.956816709;unc_s:0.0002507416498506334;after_mach:139100156292;type:3;settimeofday:0
2016-07-30 00:42:48 -0400 timed[62]: type:21;rtc_s:8670536.995452;mach:139125083863
2016-07-30 00:42:48 -0400 timed[62]: unc_ns:500938;src:13;before_mach:139124436552;after_mach:139124436697;rtc_s:8670536.968480708;type:11;t_ns:491546568400639880
2016-07-30 00:42:48 -0400 timed[62]: t_s:491546568.4008424;rateSf:0.9999896661291756;sf_var:7.930044421917708e-12;rtc_s:8670536.968480708;utc_var:6.306507062895193e-08;type:29;utc_sf_cov:1.018911368755784e-10
2016-07-30 00:42:48 -0400 timed[62]: _utc_sf_cov:8.14271809181521e-11;_rateSf:0.9999896004081723;_rtc:8670536.968480708;_utc_var:5.039899517453055e-08;type:32;_utc:491546568.4008018;_sf_var:7.896981813461407e-12
2016-07-30 00:42:48 -0400 timed[62]: type:21;rtc_s:8670536.999175999;mach:139125173952
2016-07-30 00:42:48 -0400 timed[62]: need_rtc_s:9170525.743620444;bb_want_rtc_s:inf;type:20;rtc_s:8670537.012435;want_rtc_s:8673025.743620444
2016-07-30 00:42:48 -0400 timed[62]: t_s:491546568.4294953;adjust:-9.465217590332031e-05;didSet:1;mach:139125125918;rtc_s:8670536.997174583;unc_s:0.0002245129795672289;after_mach:139125125947;type:3;settimeofday:0
2016-07-30 00:42:49 -0400 timed[62]: type:21;rtc_s:8670538.026763;mach:139149835524
2016-07-30 00:42:49 -0400 timed[62]: unc_ns:500950;src:13;before_mach:139149174144;after_mach:139149174282;rtc_s:8670537.9992055;type:11;t_ns:491546569431356926
2016-07-30 00:42:49 -0400 timed[62]: t_s:491546569.4315159;rateSf:0.9999896004081723;sf_var:7.925635962673724e-12;rtc_s:8670537.9992055;utc_var:5.057525305296774e-08;type:29;utc_sf_cov:8.958156312474948e-11
2016-07-30 00:42:49 -0400 timed[62]: _utc_sf_cov:7.455597452370418e-11;_rateSf:0.9999895531804991;_rtc:8670537.9992055;_utc_var:4.209222463439236e-08;type:32;_utc:491546569.4314892;_sf_var:7.899021845442489e-12
2016-07-30 00:42:49 -0400 timed[62]: type:21;rtc_s:8670538.039886;mach:139150149793
2016-07-30 00:42:49 -0400 timed[62]: need_rtc_s:9170527.741010649;bb_want_rtc_s:inf;type:20;rtc_s:8670538.044753;want_rtc_s:8673027.741010649
2016-07-30 00:42:49 -0400 timed[62]: t_s:491546569.4619897;adjust:1.26957893371582e-05;didSet:1;mach:139149905480;rtc_s:8670538.029706292;unc_s:0.0002051791669498296;after_mach:139149905500;type:3;settimeofday:0
2016-07-30 00:42:50 -0400 timed[62]: type:21;rtc_s:8670539.056518;mach:139174549802
2016-07-30 00:42:50 -0400 timed[62]: unc_ns:500906;src:13;before_mach:139173940885;after_mach:139173941030;rtc_s:8670539.031146457;type:11;t_ns:491546570463288795
2016-07-30 00:42:50 -0400 timed[62]: t_s:491546570.4634194;rateSf:0.9999895531804991;sf_var:7.927709804061861e-12;rtc_s:8670539.031146457;utc_var:4.225452123042699e-08;type:29;utc_sf_cov:8.272210083019466e-11
2016-07-30 00:42:50 -0400 timed[62]: _utc_sf_cov:7.079904987484393e-11;_rateSf:0.9999895163305667;_rtc:8670539.031146457;_utc_var:3.616421640658679e-08;type:32;_utc:491546570.4634005;_sf_var:7.90436792739135e-12
2016-07-30 00:42:50 -0400 timed[62]: need_rtc_s:9170529.522700684;bb_want_rtc_s:inf;type:20;rtc_s:8670539.071242999;want_rtc_s:8673029.522700684
2016-07-30 00:42:50 -0400 timed[62]: type:21;rtc_s:8670539.07315;mach:139174947516
2016-07-30 00:42:50 -0400 timed[62]: t_s:491546570.4913682;adjust:-4.76837158203125e-06;didSet:1;mach:139174610664;rtc_s:8670539.059114499;unc_s:0.0001901826901194835;after_mach:139174610678;type:3;settimeofday:0
2016-07-30 00:42:51 -0400 timed[62]: type:21;rtc_s:8670540.094177;mach:139199453895
2016-07-30 00:42:51 -0400 timed[62]: unc_ns:500931;src:13;before_mach:139198814540;after_mach:139198814717;rtc_s:8670540.067537209;type:11;t_ns:491546571499676807
2016-07-30 00:42:51 -0400 timed[62]: t_s:491546571.4997804;rateSf:0.9999895163305667;sf_var:7.933179590285837e-12;rtc_s:8670540.067537209;utc_var:3.631946781055997e-08;type:29;utc_sf_cov:7.900599376251912e-11
2016-07-30 00:42:51 -0400 timed[62]: _utc_sf_cov:6.901666616921086e-11;_rateSf:0.9999894878383226;_rtc:8670540.067537209;_utc_var:3.172732176320001e-08;type:32;_utc:491546571.4997673;_sf_var:7.911449735717341e-12
2016-07-30 00:42:51 -0400 timed[62]: need_rtc_s:9170531.161454771;bb_want_rtc_s:inf;type:20;rtc_s:8670540.114088999;want_rtc_s:8673031.161454771
2016-07-30 00:42:51 -0400 timed[62]: type:21;rtc_s:8670540.116851;mach:139199997474
2016-07-30 00:42:51 -0400 timed[62]: t_s:491546571.5294456;adjust:-7.140636444091797e-05;didSet:1;mach:139199526229;rtc_s:8670540.097215792;unc_s:0.0001781368111061053;after_mach:139199526250;type:3;settimeofday:0
2016-07-30 00:42:52 -0400 timed[62]: type:21;rtc_s:8670541.216278;mach:139226385047
2016-07-30 00:42:52 -0400 timed[62]: unc_ns:500914;src:13;before_mach:139225766272;after_mach:139225766412;rtc_s:8670541.190495709;type:11;t_ns:491546572622646297
2016-07-30 00:42:52 -0400 timed[62]: t_s:491546572.622714;rateSf:0.9999894878383226;sf_var:7.942667982015624e-12;rtc_s:8670541.190495709;utc_var:3.18923172107542e-08;type:29;utc_sf_cov:7.791842429428577e-11
2016-07-30 00:42:52 -0400 timed[62]: _utc_sf_cov:6.913154109132972e-11;_rateSf:0.9999894691828088;_rtc:8670541.190495709;_utc_var:2.829581139148709e-08;type:32;_utc:491546572.6227064;_sf_var:7.921200112838612e-12
2016-07-30 00:42:52 -0400 timed[62]: type:21;rtc_s:8670541.218688;mach:139226442888
2016-07-30 00:42:52 -0400 timed[62]: need_rtc_s:9170532.779816279;bb_want_rtc_s:inf;type:20;rtc_s:8670541.230086999;want_rtc_s:8673032.779816279
2016-07-30 00:42:52 -0400 timed[62]: t_s:491546572.6500599;adjust:-7.414817810058594e-05;didSet:1;mach:139226422765;rtc_s:8670541.217849541;unc_s:0.0001682278551179203;after_mach:139226422779;type:3;settimeofday:0
2016-07-30 00:42:53 -0400 timed[62]: type:21;rtc_s:8670542.362243;mach:139253889043
2016-07-30 00:42:53 -0400 timed[62]: unc_ns:501164;src:13;before_mach:139252970424;after_mach:139252970562;rtc_s:8670542.323967209;type:11;t_ns:491546573756136204
2016-07-30 00:42:53 -0400 timed[62]: t_s:491546573.7561659;rateSf:0.9999894691828088;sf_var:7.952710620542184e-12;rtc_s:8670542.323967209;utc_var:2.846271897149189e-08;type:29;utc_sf_cov:7.81278537972653e-11
2016-07-30 00:42:53 -0400 timed[62]: _utc_sf_cov:7.017540823615636e-11;_rateSf:0.9999894608727333;_rtc:8670542.323967209;_utc_var:2.556556754417526e-08;type:32;_utc:491546573.7561629;_sf_var:7.930881802826233e-12
2016-07-30 00:42:53 -0400 timed[62]: need_rtc_s:9170534.329349091;bb_want_rtc_s:inf;type:20;rtc_s:8670542.381149;want_rtc_s:8673034.329349091
2016-07-30 00:42:53 -0400 timed[62]: type:21;rtc_s:8670542.383575;mach:139254399177
2016-07-30 00:42:53 -0400 timed[62]: t_s:491546573.7989075;adjust:2.551078796386719e-05;didSet:1;mach:139253994472;rtc_s:8670542.366712291;unc_s:0.0001599147129197911;after_mach:139253994495;type:3;settimeofday:0
2016-07-30 00:43:59 -0400 timed[62]: type:21;rtc_s:8670607.766052;mach:140823581772
2016-07-30 00:43:59 -0400 timed[62]: unc_ns:500908;src:13;before_mach:140822970290;after_mach:140822970433;rtc_s:8670607.740573583;type:11;t_ns:491546639171904327
2016-07-30 00:43:59 -0400 timed[62]: t_s:491546639.1720798;rateSf:0.9999894608727333;sf_var:9.749463460025795e-12;rtc_s:8670607.740573583;utc_var:7.127982616530815e-08;type:29;utc_sf_cov:6.484695015451105e-10
2016-07-30 00:43:59 -0400 timed[62]: _utc_sf_cov:5.050047594772395e-10;_rateSf:0.9999891076924426;_rtc:8670607.740573583;_utc_var:5.5510168762635e-08;type:32;_utc:491546639.172041;_sf_var:8.444290386759673e-12
2016-07-30 00:43:59 -0400 timed[62]: type:21;rtc_s:8670607.768981;mach:140823650805
2016-07-30 00:43:59 -0400 timed[62]: need_rtc_s:9170595.960275544;bb_want_rtc_s:inf;type:20;rtc_s:8670607.784103001;want_rtc_s:8673095.960275544
2016-07-30 00:43:59 -0400 timed[62]: t_s:491546639.200096;adjust:-0.0008070468902587891;didSet:1;mach:140823642353;rtc_s:8670607.768628834;unc_s:0.0002356720626628887;after_mach:140823642373;type:3;settimeofday:0
2016-07-30 00:44:00 -0400 timed[62]: type:21;rtc_s:8670608.887267999;mach:140850490167
2016-07-30 00:44:00 -0400 timed[62]: unc_ns:500944;src:13;before_mach:140849835177;after_mach:140849835337;rtc_s:8670608.85997675;type:11;t_ns:491546640291261395
2016-07-30 00:44:00 -0400 timed[62]: t_s:491546640.291432;rateSf:0.9999891076924426;sf_var:8.475409794793833e-12;rtc_s:8670608.85997675;utc_var:5.665137084638532e-08;type:29;utc_sf_cov:5.14474742458632e-10
2016-07-30 00:44:00 -0400 timed[62]: _utc_sf_cov:4.197218968956308e-10;_rateSf:0.9999888223728657;_rtc:8670608.85997675;_utc_var:4.621766409707589e-08;type:32;_utc:491546640.2914006;_sf_var:7.614919658545735e-12
2016-07-30 00:44:00 -0400 timed[62]: type:21;rtc_s:8670608.890869001;mach:140850578396
2016-07-30 00:44:00 -0400 timed[62]: need_rtc_s:9170598.110829795;bb_want_rtc_s:inf;type:20;rtc_s:8670608.906265;want_rtc_s:8673098.110829795
2016-07-30 00:44:00 -0400 timed[62]: t_s:491546640.3212026;adjust:-0.0001374483108520508;didSet:1;mach:140850552239;rtc_s:8670608.889779126;unc_s:0.0002150437868143224;after_mach:140850552261;type:3;settimeofday:0
2016-07-30 00:44:01 -0400 timed[62]: type:21;rtc_s:8670609.998016;mach:140877149673
2016-07-30 00:44:01 -0400 timed[62]: unc_ns:500922;src:13;before_mach:140876521306;after_mach:140876521478;rtc_s:8670609.971834041;type:11;t_ns:491546641403169205
2016-07-30 00:44:01 -0400 timed[62]: t_s:491546641.4032454;rateSf:0.9999888223728657;sf_var:7.645829291244182e-12;rtc_s:8670609.971834041;utc_var:4.716043230346698e-08;type:29;utc_sf_cov:4.282057843909763e-10
2016-07-30 00:44:01 -0400 timed[62]: _utc_sf_cov:3.60458521400208e-10;_rateSf:0.9999887129457544;_rtc:8670609.971834041;_utc_var:3.969908935461953e-08;type:32;_utc:491546641.4032333;_sf_var:7.030699860228109e-12
2016-07-30 00:44:01 -0400 timed[62]: type:21;rtc_s:8670610.012755999;mach:140877503166
2016-07-30 00:44:01 -0400 timed[62]: need_rtc_s:9170600.009518879;bb_want_rtc_s:inf;type:20;rtc_s:8670610.021957001;want_rtc_s:8673100.009518879
2016-07-30 00:44:01 -0400 timed[62]: t_s:491546641.4364635;adjust:-3.457069396972656e-06;didSet:1;mach:140877318572;rtc_s:8670610.005064582;unc_s:0.0001993094797263936;after_mach:140877318597;type:3;settimeofday:0
2016-07-30 00:44:02 -0400 timed[62]: type:21;rtc_s:8670611.082611;mach:140903180783
2016-07-30 00:44:02 -0400 timed[62]: unc_ns:500617;src:13;before_mach:140902919156;after_mach:140902919340;rtc_s:8670611.071709875;type:11;t_ns:491546642503065525
2016-07-30 00:44:02 -0400 timed[62]: t_s:491546642.5030968;rateSf:0.9999887129457544;sf_var:7.061276408408838e-12;rtc_s:8670611.071709875;utc_var:4.050052614434986e-08;type:29;utc_sf_cov:3.682082334746625e-10
2016-07-30 00:44:02 -0400 timed[62]: _utc_sf_cov:3.169828513103535e-10;_rateSf:0.999988673442222;_rtc:8670611.071709875;_utc_var:3.486606514921659e-08;type:32;_utc:491546642.5030924;_sf_var:6.595563755477882e-12
2016-07-30 00:44:02 -0400 timed[62]: type:21;rtc_s:8670611.087158;mach:140903289142
2016-07-30 00:44:02 -0400 timed[62]: need_rtc_s:9170601.735481361;bb_want_rtc_s:inf;type:20;rtc_s:8670611.103744;want_rtc_s:8673101.735481361
2016-07-30 00:44:02 -0400 timed[62]: t_s:491546642.5164573;adjust:-3.069639205932617e-05;didSet:1;mach:140903239149;rtc_s:8670611.085074959;unc_s:0.0001867518719830459;after_mach:140903239175;type:3;settimeofday:0
2016-07-30 00:44:03 -0400 timed[62]: type:21;rtc_s:8670612.131163999;mach:140928346240
2016-07-30 00:44:03 -0400 timed[62]: unc_ns:500923;src:13;before_mach:140927716750;after_mach:140927716896;rtc_s:8670612.104935249;type:11;t_ns:491546643536283454
2016-07-30 00:44:03 -0400 timed[62]: t_s:491546643.5363061;rateSf:0.999988673442222;sf_var:6.62428742088332e-12;rtc_s:8670612.104935249;utc_var:3.55281459458744e-08;type:29;utc_sf_cov:3.238123941502721e-10
2016-07-30 00:44:03 -0400 timed[62]: _utc_sf_cov:2.836505836266287e-10;_rateSf:0.9999886478384212;_rtc:8670612.104935249;_utc_var:3.112166030322669e-08;type:32;_utc:491546643.5363033;_sf_var:6.258242650664155e-12
2016-07-30 00:44:03 -0400 timed[62]: type:21;rtc_s:8670612.135589;mach:140928452089
2016-07-30 00:44:03 -0400 timed[62]: need_rtc_s:9170603.284269132;bb_want_rtc_s:inf;type:20;rtc_s:8670612.148390999;want_rtc_s:8673103.284269132
2016-07-30 00:44:03 -0400 timed[62]: t_s:491546643.5649951;adjust:-4.088878631591797e-05;didSet:1;mach:140928405011;rtc_s:8670612.133627417;unc_s:0.0001764601524932621;after_mach:140928405039;type:3;settimeofday:0
2016-07-30 00:44:09 -0400 timed[62]: type:21;rtc_s:8670617.681427;mach:141061548381
2016-07-30 00:44:09 -0400 timed[62]: unc_ns:500935;src:13;before_mach:141060904940;after_mach:141060905070;rtc_s:8670617.654616958;type:11;t_ns:491546649085717681
2016-07-30 00:44:09 -0400 timed[62]: t_s:491546649.085922;rateSf:0.9999886478384212;sf_var:6.412523802152578e-12;rtc_s:8670617.654616958;utc_var:3.446433252348344e-08;type:29;utc_sf_cov:3.188099440327699e-10
2016-07-30 00:44:09 -0400 timed[62]: _utc_sf_cov:2.803111485980486e-10;_rateSf:0.9999884195950429;_rtc:8670617.654616958;_utc_var:3.030249468733546e-08;type:32;_utc:491546649.0858973;_sf_var:6.056393335691074e-12
2016-07-30 00:44:09 -0400 timed[62]: type:21;rtc_s:8670617.683441;mach:141061597146
2016-07-30 00:44:09 -0400 timed[62]: need_rtc_s:9170608.950811082;bb_want_rtc_s:inf;type:20;rtc_s:8670617.692977;want_rtc_s:8673108.950811082
2016-07-30 00:44:09 -0400 timed[62]: t_s:491546649.1140797;adjust:5.263090133666992e-05;didSet:1;mach:141061581753;rtc_s:8670617.682799624;unc_s:0.0001741240381845758;after_mach:141061581760;type:3;settimeofday:0
2016-07-30 00:44:10 -0400 timed[62]: type:21;rtc_s:8670618.806120999;mach:141088542459
2016-07-30 00:44:10 -0400 timed[62]: unc_ns:500906;src:13;before_mach:141087933027;after_mach:141087933138;rtc_s:8670618.780727999;type:11;t_ns:491546650211872779
2016-07-30 00:44:10 -0400 timed[62]: t_s:491546650.2119953;rateSf:0.9999884195950429;sf_var:6.087699222651848e-12;rtc_s:8670618.780727999;utc_var:3.09415111500621e-08;type:29;utc_sf_cov:2.871489469590678e-10
2016-07-30 00:44:10 -0400 timed[62]: _utc_sf_cov:2.556256268732142e-10;_rateSf:0.999988294743577;_rtc:8670618.780727999;_utc_var:2.754474034434344e-08;type:32;_utc:491546650.2119818;_sf_var:5.795150868325969e-12
2016-07-30 00:44:10 -0400 timed[62]: type:21;rtc_s:8670618.80957;mach:141088623822
2016-07-30 00:44:10 -0400 timed[62]: need_rtc_s:9170610.482423903;bb_want_rtc_s:inf;type:20;rtc_s:8670618.825576;want_rtc_s:8673110.482423903
2016-07-30 00:44:10 -0400 timed[62]: t_s:491546650.2400656;adjust:-8.344650268554688e-06;didSet:1;mach:141088605633;rtc_s:8670618.808812125;unc_s:0.0001660125417564925;after_mach:141088605642;type:3;settimeofday:0
2016-07-30 00:44:11 -0400 timed[62]: type:21;rtc_s:8670619.916595001;mach:141115193123
2016-07-30 00:44:11 -0400 timed[62]: unc_ns:500917;src:13;before_mach:141114570762;after_mach:141114570868;rtc_s:8670619.890663292;type:11;t_ns:491546651321764335
2016-07-30 00:44:11 -0400 timed[62]: t_s:491546651.3219041;rateSf:0.999988294743577;sf_var:5.826007069470649e-12;rtc_s:8670619.890663292;utc_var:2.811934819855627e-08;type:29;utc_sf_cov:2.620749935434076e-10
2016-07-30 00:44:11 -0400 timed[62]: _utc_sf_cov:2.356650327685186e-10;_rateSf:0.9999881634675399;_rtc:8670619.890663292;_utc_var:2.528569027149306e-08;type:32;_utc:491546651.3218901;_sf_var:5.579863728924938e-12
2016-07-30 00:44:11 -0400 timed[62]: type:21;rtc_s:8670619.918853;mach:141115246702
2016-07-30 00:44:11 -0400 timed[62]: need_rtc_s:9170611.939925862;bb_want_rtc_s:inf;type:20;rtc_s:8670619.927657001;want_rtc_s:8673111.939925862
2016-07-30 00:44:11 -0400 timed[62]: t_s:491546651.3488813;adjust:-3.069639205932617e-05;didSet:1;mach:141115217947;rtc_s:8670619.917654876;unc_s:0.000159057719246775;after_mach:141115217955;type:3;settimeofday:0
2016-07-30 00:44:12 -0400 timed[62]: type:21;rtc_s:8670621.040100001;mach:141142157869
2016-07-30 00:44:12 -0400 timed[62]: unc_ns:500904;src:13;before_mach:141141551645;after_mach:141141551748;rtc_s:8670621.014840668;type:11;t_ns:491546652445953623
2016-07-30 00:44:12 -0400 timed[62]: t_s:491546652.4460541;rateSf:0.9999881634675399;sf_var:5.61111585997241e-12;rtc_s:8670621.014840668;utc_var:2.582261372393971e-08;type:29;utc_sf_cov:2.419553558020044e-10
2016-07-30 00:44:12 -0400 timed[62]: _utc_sf_cov:2.193774875800661e-10;_rateSf:0.9999880756016303;_rtc:8670621.014840668;_utc_var:2.341299742149167e-08;type:32;_utc:491546652.4460447;_sf_var:5.399563451014469e-12
2016-07-30 00:44:12 -0400 timed[62]: type:21;rtc_s:8670621.042967999;mach:141142226689
2016-07-30 00:44:12 -0400 timed[62]: need_rtc_s:9170613.364187522;bb_want_rtc_s:inf;type:20;rtc_s:8670621.054091999;want_rtc_s:8673113.364187522
2016-07-30 00:44:12 -0400 timed[62]: t_s:491546652.4725683;adjust:-7.367134094238281e-05;didSet:1;mach:141142188207;rtc_s:8670621.041364582;unc_s:0.0001530533370850486;after_mach:141142188216;type:3;settimeofday:0
2016-07-30 00:44:13 -0400 timed[62]: type:21;rtc_s:8670622.137390001;mach:141168492091
2016-07-30 00:44:13 -0400 timed[62]: unc_ns:501044;src:13;before_mach:141167717497;after_mach:141167717667;rtc_s:8670622.105115252;type:11;t_ns:491546653536187482
2016-07-30 00:44:13 -0400 timed[62]: t_s:491546653.5363063;rateSf:0.9999880756016303;sf_var:5.429873084437112e-12;rtc_s:8670622.105115252;utc_var:2.389779128286722e-08;type:29;utc_sf_cov:2.252810172844422e-10
2016-07-30 00:44:13 -0400 timed[62]: _utc_sf_cov:2.056998264776239e-10;_rateSf:0.999987978217858;_rtc:8670622.105115252;_utc_var:2.182062021620559e-08;type:32;_utc:491546653.536296;_sf_var:5.245284034663156e-12
2016-07-30 00:44:13 -0400 timed[62]: type:21;rtc_s:8670622.143127;mach:141168631244
2016-07-30 00:44:13 -0400 timed[62]: need_rtc_s:9170614.719213104;bb_want_rtc_s:inf;type:20;rtc_s:8670622.158096001;want_rtc_s:8673114.719213104
2016-07-30 00:44:13 -0400 timed[62]: t_s:491546653.573317;adjust:-5.501508712768555e-05;didSet:1;mach:141168607476;rtc_s:8670622.142136667;unc_s:0.0001477724890588343;after_mach:141168607499;type:3;settimeofday:0
2016-07-30 00:44:14 -0400 timed[62]: type:21;rtc_s:8670623.250488;mach:141195209293
2016-07-30 00:44:14 -0400 timed[62]: unc_ns:500974;src:13;before_mach:141194518499;after_mach:141194518674;rtc_s:8670623.221704917;type:11;t_ns:491546654652882299
2016-07-30 00:44:14 -0400 timed[62]: t_s:491546654.6528723;rateSf:0.999987978217858;sf_var:5.276325227361635e-12;rtc_s:8670623.221704917;utc_var:2.228653739327377e-08;type:29;utc_sf_cov:2.115739865603687e-10
2016-07-30 00:44:14 -0400 timed[62]: _utc_sf_cov:1.943185723738799e-10;_rateSf:0.9999879859709007;_rtc:8670623.221704917;_utc_var:2.046890640869202e-08;type:32;_utc:491546654.6528731;_sf_var:5.112513473890788e-12
2016-07-30 00:44:14 -0400 timed[62]: type:21;rtc_s:8670623.254210999;mach:141195297778
2016-07-30 00:44:14 -0400 timed[62]: need_rtc_s:9170616.068225622;bb_want_rtc_s:inf;type:20;rtc_s:8670623.270187;want_rtc_s:8673116.068225622
2016-07-30 00:44:14 -0400 timed[62]: t_s:491546654.6834044;adjust:-3.55839729309082e-05;didSet:1;mach:141195250393;rtc_s:8670623.252236623;unc_s:0.0001431143761159586;after_mach:141195250423;type:3;settimeofday:0
2016-07-30 00:44:15 -0400 timed[62]: type:21;rtc_s:8670624.354217;mach:141221696335
2016-07-30 00:44:15 -0400 timed[62]: unc_ns:501045;src:13;before_mach:141220920027;after_mach:141220920204;rtc_s:8670624.321870834;type:11;t_ns:491546655752932439
2016-07-30 00:44:15 -0400 timed[62]: t_s:491546655.7530258;rateSf:0.9999879859709007;sf_var:5.143098086372457e-12;rtc_s:8670624.321870834;utc_var:2.090267209638226e-08;type:29;utc_sf_cov:1.9996000952016e-10
2016-07-30 00:44:15 -0400 timed[62]: _utc_sf_cov:1.845906280700969e-10;_rateSf:0.9999879173389341;_rtc:8670624.321870834;_utc_var:1.929604514359399e-08;type:32;_utc:491546655.7530186;_sf_var:4.996070871531083e-12
2016-07-30 00:44:15 -0400 timed[62]: type:21;rtc_s:8670624.371246001;mach:141222107802
2016-07-30 00:44:15 -0400 timed[62]: need_rtc_s:9170617.376360567;bb_want_rtc_s:inf;type:20;rtc_s:8670624.378371;want_rtc_s:8673117.376360567
2016-07-30 00:44:15 -0400 timed[62]: t_s:491546655.7898802;adjust:-6.979703903198242e-05;didSet:1;mach:141221807487;rtc_s:8670624.358732875;unc_s:0.0001389651773684254;after_mach:141221807513;type:3;settimeofday:0
2016-07-30 00:44:16 -0400 timed[62]: type:21;rtc_s:8670625.454162;mach:141248095157
2016-07-30 00:44:16 -0400 timed[62]: unc_ns:501038;src:13;before_mach:141247327313;after_mach:141247327488;rtc_s:8670625.422168499;type:11;t_ns:491546656853222254
2016-07-30 00:44:16 -0400 timed[62]: t_s:491546656.853303;rateSf:0.9999879173389341;sf_var:5.026659146624296e-12;rtc_s:8670625.422168499;utc_var:1.970831527965987e-08;type:29;utc_sf_cov:1.901046212891678e-10
2016-07-30 00:44:16 -0400 timed[62]: _utc_sf_cov:1.762665203543982e-10;_rateSf:0.9999878606306649;_rtc:8670625.422168499;_utc_var:1.827370704002453e-08;type:32;_utc:491546656.8532971;_sf_var:4.893178080371988e-12
2016-07-30 00:44:16 -0400 timed[62]: type:21;rtc_s:8670625.492094999;mach:141249006904
2016-07-30 00:44:16 -0400 timed[62]: need_rtc_s:9170618.663154695;bb_want_rtc_s:inf;type:20;rtc_s:8670625.493453;want_rtc_s:8673118.663154695
2016-07-30 00:44:16 -0400 timed[62]: t_s:491546656.8891013;adjust:3.325939178466797e-05;didSet:1;mach:141248187978;rtc_s:8670625.457973082;unc_s:0.0001352294848231344;after_mach:141248188000;type:3;settimeofday:0
2016-07-30 00:44:17 -0400 timed[62]: type:21;rtc_s:8670626.554504;mach:141274503684
2016-07-30 00:44:17 -0400 timed[62]: unc_ns:501052;src:13;before_mach:141273719848;after_mach:141273720020;rtc_s:8670626.521844165;type:11;t_ns:491546657952897352
2016-07-30 00:44:17 -0400 timed[62]: t_s:491546657.9529594;rateSf:0.9999878606306649;sf_var:4.923749063901825e-12;rtc_s:8670626.521844165;utc_var:1.866730862411523e-08;type:29;utc_sf_cov:1.816642383047794e-10
2016-07-30 00:44:17 -0400 timed[62]: _utc_sf_cov:1.690912933892357e-10;_rateSf:0.9999878188393401;_rtc:8670626.521844165;_utc_var:1.737534799805798e-08;type:32;_utc:491546657.9529551;_sf_var:4.801393211361863e-12
2016-07-30 00:44:18 -0400 timed[62]: need_rtc_s:9170619.931065001;bb_want_rtc_s:inf;type:20;rtc_s:8670626.577834001;want_rtc_s:8673119.931065001
2016-07-30 00:44:18 -0400 timed[62]: type:21;rtc_s:8670626.580352001;mach:141275124003
2016-07-30 00:44:18 -0400 timed[62]: t_s:491546657.9900867;adjust:2.6702880859375e-05;didSet:1;mach:141274610984;rtc_s:8670626.558976209;unc_s:0.0001318666987076477;after_mach:141274611008;type:3;settimeofday:0
2016-07-30 00:44:19 -0400 timed[62]: type:21;rtc_s:8670627.671875;mach:141301318966
2016-07-30 00:44:19 -0400 timed[62]: unc_ns:501047;src:13;before_mach:141300541183;after_mach:141300541300;rtc_s:8670627.639467375;type:11;t_ns:491546659070436313
2016-07-30 00:44:19 -0400 timed[62]: t_s:491546659.0705647;rateSf:0.9999878188393401;sf_var:4.832463136598565e-12;rtc_s:8670627.639467375;utc_var:1.775931897463868e-08;type:29;utc_sf_cov:1.74474804117154e-10
2016-07-30 00:44:19 -0400 timed[62]: _utc_sf_cov:1.629477711456731e-10;_rateSf:0.9999877354676499;_rtc:8670627.639467375;_utc_var:1.658601342827309e-08;type:32;_utc:491546659.0705563;_sf_var:4.719216856005082e-12
2016-07-30 00:44:19 -0400 timed[62]: type:21;rtc_s:8670627.674040999;mach:141301371957
2016-07-30 00:44:19 -0400 timed[62]: need_rtc_s:9170621.200132506;bb_want_rtc_s:inf;type:20;rtc_s:8670627.686996;want_rtc_s:8673121.200132506
2016-07-30 00:44:19 -0400 timed[62]: t_s:491546659.1048346;adjust:2.622604370117188e-06;didSet:1;mach:141301364880;rtc_s:8670627.673746124;unc_s:0.0001288328578469344;after_mach:141301364892;type:3;settimeofday:0
2016-07-30 00:44:43 -0400 timed[62]: type:21;rtc_s:8670652.204376001;mach:141890102419
2016-07-30 00:44:43 -0400 timed[62]: unc_ns:500962;src:13;before_mach:141889426006;after_mach:141889426123;rtc_s:8670652.176192125;type:11;t_ns:491546683606982209
2016-07-30 00:44:43 -0400 timed[62]: t_s:491546683.6069801;rateSf:0.9999877354676499;sf_var:5.401337804053763e-12;rtc_s:8670652.176192125;utc_var:2.756052172172593e-08;type:29;utc_sf_cov:2.871104031510299e-10
2016-07-30 00:44:43 -0400 timed[62]: _utc_sf_cov:2.587002520553994e-10;_rateSf:0.9999877376795663;_rtc:8670652.176192125;_utc_var:2.483335273796482e-08;type:32;_utc:491546683.6069803;_sf_var:5.105376427959126e-12
2016-07-30 00:44:43 -0400 timed[62]: type:21;rtc_s:8670652.209106;mach:141890215329
2016-07-30 00:44:43 -0400 timed[62]: need_rtc_s:9170644.296891278;bb_want_rtc_s:inf;type:20;rtc_s:8670652.22383;want_rtc_s:8673144.296891278
2016-07-30 00:44:43 -0400 timed[62]: t_s:491546683.6373448;adjust:-0.000374138355255127;didSet:1;mach:141890154153;rtc_s:8670652.206557;unc_s:0.0001576437111605925;after_mach:141890154172;type:3;settimeofday:0
2016-07-30 00:44:44 -0400 timed[62]: type:21;rtc_s:8670653.287872;mach:141916104636
2016-07-30 00:44:44 -0400 timed[62]: unc_ns:500967;src:13;before_mach:141915422642;after_mach:141915422795;rtc_s:8670653.259455582;type:11;t_ns:491546684690162556
2016-07-30 00:44:44 -0400 timed[62]: t_s:491546684.6902305;rateSf:0.9999877376795663;sf_var:5.135491152058762e-12;rtc_s:8670653.259455582;utc_var:2.539983652965866e-08;type:29;utc_sf_cov:2.642470308631898e-10
2016-07-30 00:44:44 -0400 timed[62]: _utc_sf_cov:2.399611911692651e-10;_rateSf:0.9999876727104203;_rtc:8670653.259455582;_utc_var:2.306544376014991e-08;type:32;_utc:491546684.6902242;_sf_var:4.882833579887165e-12
2016-07-30 00:44:44 -0400 timed[62]: type:21;rtc_s:8670653.292358;mach:141916213029
2016-07-30 00:44:44 -0400 timed[62]: need_rtc_s:9170645.665799716;bb_want_rtc_s:inf;type:20;rtc_s:8670653.319076;want_rtc_s:8673145.665799716
2016-07-30 00:44:44 -0400 timed[62]: t_s:491546684.7216401;adjust:-5.960464477539062e-06;didSet:1;mach:141916177361;rtc_s:8670653.290871833;unc_s:0.0001519254676559671;after_mach:141916177381;type:3;settimeofday:0
2016-07-30 00:44:45 -0400 timed[62]: type:21;rtc_s:8670654.411590001;mach:141943076143
2016-07-30 00:44:45 -0400 timed[62]: unc_ns:500937;src:13;before_mach:141942430263;after_mach:141942430392;rtc_s:8670654.384678334;type:11;t_ns:491546685815463260
2016-07-30 00:44:45 -0400 timed[62]: t_s:491546685.8154331;rateSf:0.9999876727104203;sf_var:4.914114772389172e-12;rtc_s:8670654.384678334;utc_var:2.361165882959633e-08;type:29;utc_sf_cov:2.454730657617071e-10
2016-07-30 00:44:45 -0400 timed[62]: _utc_sf_cov:2.243620436377293e-10;_rateSf:0.9999876996762249;_rtc:8670654.384678334;_utc_var:2.158102361351397e-08;type:32;_utc:491546685.8154357;_sf_var:4.694638988856319e-12
2016-07-30 00:44:45 -0400 timed[62]: type:21;rtc_s:8670654.415588001;mach:141943172328
2016-07-30 00:44:45 -0400 timed[62]: need_rtc_s:9170647.039437763;bb_want_rtc_s:inf;type:20;rtc_s:8670654.427901;want_rtc_s:8673147.039437763
2016-07-30 00:44:45 -0400 timed[62]: t_s:491546685.8443526;adjust:-8.642673492431641e-05;didSet:1;mach:141943124510;rtc_s:8670654.413595583;unc_s:0.0001469547379753094;after_mach:141943124529;type:3;settimeofday:0
2016-07-30 00:44:46 -0400 timed[62]: type:21;rtc_s:8670655.545073999;mach:141970276412
2016-07-30 00:44:46 -0400 timed[62]: unc_ns:500621;src:13;before_mach:141970008877;after_mach:141970009026;rtc_s:8670655.533926707;type:11;t_ns:491546686964558187
2016-07-30 00:44:46 -0400 timed[62]: t_s:491546686.9646699;rateSf:0.9999876996762249;sf_var:4.726588093619141e-12;rtc_s:8670655.533926707;utc_var:2.210293365335315e-08;type:29;utc_sf_cov:2.297757085847139e-10
2016-07-30 00:44:46 -0400 timed[62]: _utc_sf_cov:2.111536083633434e-10;_rateSf:0.9999876055175982;_rtc:8670655.533926707;_utc_var:2.031160832913031e-08;type:32;_utc:491546686.9646609;_sf_var:4.532998123128874e-12
2016-07-30 00:44:46 -0400 timed[62]: type:21;rtc_s:8670655.554656001;mach:141970507336
2016-07-30 00:44:46 -0400 timed[62]: need_rtc_s:9170648.407986712;bb_want_rtc_s:inf;type:20;rtc_s:8670655.558577999;want_rtc_s:8673148.407986712
2016-07-30 00:44:47 -0400 timed[62]: t_s:491546686.9785314;adjust:8.642673492431641e-05;didSet:1;mach:141970342730;rtc_s:8670655.547797417;unc_s:0.0001425459893649643;after_mach:141970342739;type:3;settimeofday:0
2016-07-30 00:44:48 -0400 timed[62]: type:21;rtc_s:8670656.601988999;mach:141995646753
2016-07-30 00:44:48 -0400 timed[62]: unc_ns:500949;src:13;before_mach:141994986016;after_mach:141994986189;rtc_s:8670656.574458292;type:11;t_ns:491546688005259799
2016-07-30 00:44:48 -0400 timed[62]: t_s:491546688.0051796;rateSf:0.9999876055175982;sf_var:4.561924901191679e-12;rtc_s:8670656.574458292;utc_var:2.075595067060915e-08;type:29;utc_sf_cov:2.158853856982862e-10
2016-07-30 00:44:48 -0400 timed[62]: _utc_sf_cov:1.993936677349293e-10;_rateSf:0.9999876692629369;_rtc:8670656.574458292;_utc_var:1.917038116383659e-08;type:32;_utc:491546688.0051857;_sf_var:4.390392363374071e-12
2016-07-30 00:44:48 -0400 timed[62]: need_rtc_s:9170649.651601013;bb_want_rtc_s:inf;type:20;rtc_s:8670656.637069;want_rtc_s:8673149.651601013
2016-07-30 00:44:48 -0400 timed[62]: type:21;rtc_s:8670656.638823999;mach:141996526763
2016-07-30 00:44:48 -0400 timed[62]: t_s:491546688.03693;adjust:1.895427703857422e-05;didSet:1;mach:141995743857;rtc_s:8670656.606202915;unc_s:0.0001385071554482175;after_mach:141995743877;type:3;settimeofday:0
2016-07-30 00:46:11 -0400 timed[62]: type:21;rtc_s:8670739.648375999;mach:143988757799
2016-07-30 00:46:11 -0400 timed[62]: unc_ns:500469;src:13;before_mach:143988672912;after_mach:143988673041;rtc_s:8670739.644839041;type:11;t_ns:491546771074446594
2016-07-30 00:46:11 -0400 timed[62]: t_s:491546771.0745422;rateSf:0.9999876692629369;sf_var:6.6997489482013e-12;rtc_s:8670739.644839041;utc_var:8.790657542838997e-08;type:29;utc_sf_cov:6.600247983923226e-10
2016-07-30 00:46:11 -0400 timed[62]: _utc_sf_cov:4.885574722622001e-10;_rateSf:0.9999874828939518;_rtc:8670739.644839041;_utc_var:6.506939495475204e-08;type:32;_utc:491546771.0745174;_sf_var:5.412328902449468e-12
2016-07-30 00:46:11 -0400 timed[62]: type:21;rtc_s:8670739.651489001;mach:143988832336
2016-07-30 00:46:11 -0400 timed[62]: need_rtc_s:9170726.890487337;bb_want_rtc_s:inf;type:20;rtc_s:8670739.660475999;want_rtc_s:8673226.890487337
2016-07-30 00:46:11 -0400 timed[62]: t_s:491546771.079266;adjust:-0.001115024089813232;didSet:1;mach:143988786705;rtc_s:8670739.649587709;unc_s:0.0002550984567413106;after_mach:143988786712;type:3;settimeofday:0
2016-07-30 00:46:12 -0400 timed[62]: type:21;rtc_s:8670740.737426;mach:144014896326
2016-07-30 00:46:12 -0400 timed[62]: unc_ns:500915;src:13;before_mach:144014277042;after_mach:144014277184;rtc_s:8670740.711622499;type:11;t_ns:491546772141272360
2016-07-30 00:46:12 -0400 timed[62]: t_s:491546772.1412874;rateSf:0.9999874828939518;sf_var:5.441985482581715e-12;rtc_s:8670740.711622499;utc_var:6.611793564025859e-08;type:29;utc_sf_cov:4.943470737791115e-10
2016-07-30 00:46:12 -0400 timed[62]: _utc_sf_cov:3.91250168731149e-10;_rateSf:0.9999874593799272;_rtc:8670740.711622499;_utc_var:5.23289301131077e-08;type:32;_utc:491546772.1412843;_sf_var:4.671156011990823e-12
2016-07-30 00:46:12 -0400 timed[62]: type:21;rtc_s:8670740.744201001;mach:144015057909
2016-07-30 00:46:12 -0400 timed[62]: need_rtc_s:9170729.273863746;bb_want_rtc_s:inf;type:20;rtc_s:8670740.752517;want_rtc_s:8673229.273863746
2016-07-30 00:46:12 -0400 timed[62]: t_s:491546772.1701857;adjust:-3.826618194580078e-05;didSet:1;mach:144014969669;rtc_s:8670740.740524335;unc_s:0.0002288090973682935;after_mach:144014969703;type:3;settimeofday:0
2016-07-30 00:46:13 -0400 timed[62]: type:21;rtc_s:8670741.866485;mach:144041992523
2016-07-30 00:46:13 -0400 timed[62]: unc_ns:501128;src:13;before_mach:144041116656;after_mach:144041116820;rtc_s:8670741.829990542;type:11;t_ns:491546773259576221
2016-07-30 00:46:13 -0400 timed[62]: t_s:491546773.2596383;rateSf:0.9999874593799272;sf_var:4.702246643576019e-12;rtc_s:8670741.829990542;utc_var:5.320990888078376e-08;type:29;utc_sf_cov:3.964916257215073e-10
2016-07-30 00:46:13 -0400 timed[62]: _utc_sf_cov:3.271702146184091e-10;_rateSf:0.9999873784661076;_rtc:8670741.829990542;_utc_var:4.390684740610323e-08;type:32;_utc:491546773.2596275;_sf_var:4.185700767747571e-12
2016-07-30 00:46:13 -0400 timed[62]: need_rtc_s:9170731.353010131;bb_want_rtc_s:inf;type:20;rtc_s:8670741.895888999;want_rtc_s:8673231.353010131
2016-07-30 00:46:13 -0400 timed[62]: type:21;rtc_s:8670741.898193;mach:144042752988
2016-07-30 00:46:13 -0400 timed[62]: t_s:491546773.3108701;adjust:5.125999450683594e-06;didSet:1;mach:144042345969;rtc_s:8670741.881233875;unc_s:0.0002096223374016903;after_mach:144042345981;type:3;settimeofday:0
2016-07-30 00:46:14 -0400 timed[62]: type:21;rtc_s:8670742.906982001;mach:144066974614
2016-07-30 00:46:14 -0400 timed[62]: unc_ns:500956;src:13;before_mach:144066306032;after_mach:144066318834;rtc_s:8670742.879124418;type:11;t_ns:491546774309116408
2016-07-30 00:46:14 -0400 timed[62]: t_s:491546774.3087481;rateSf:0.9999873784661076;sf_var:4.214866689509783e-12;rtc_s:8670742.879124418;utc_var:4.459795594290819e-08;type:29;utc_sf_cov:3.315768745683473e-10
2016-07-30 00:46:14 -0400 timed[62]: _utc_sf_cov:2.815433791066711e-10;_rateSf:0.9999877917176373;_rtc:8670742.879124418;_utc_var:3.786832008041205e-08;type:32;_utc:491546774.3088037;_sf_var:3.842877666735965e-12
2016-07-30 00:46:14 -0400 timed[62]: type:21;rtc_s:8670742.911590001;mach:144067076750
2016-07-30 00:46:14 -0400 timed[62]: need_rtc_s:9170733.149232322;bb_want_rtc_s:inf;type:20;rtc_s:8670742.926467;want_rtc_s:8673233.149232322
2016-07-30 00:46:14 -0400 timed[62]: t_s:491546774.3402385;adjust:-4.953145980834961e-05;didSet:1;mach:144067052020;rtc_s:8670742.910559583;unc_s:0.0001946471408166835;after_mach:144067052041;type:3;settimeofday:0
2016-07-30 00:46:15 -0400 timed[62]: type:21;rtc_s:8670744.035858;mach:144094060240
2016-07-30 00:46:15 -0400 timed[62]: unc_ns:501022;src:13;before_mach:144093312006;after_mach:144093312184;rtc_s:8670744.004681583;type:11;t_ns:491546775434354801
2016-07-30 00:46:15 -0400 timed[62]: t_s:491546775.4343471;rateSf:0.9999877917176373;sf_var:3.874168155939448e-12;rtc_s:8670744.004681583;utc_var:3.850698809048157e-08;type:29;utc_sf_cov:2.858863672181316e-10
2016-07-30 00:46:15 -0400 timed[62]: _utc_sf_cov:2.478640497739523e-10;_rateSf:0.9999877993098584;_rtc:8670744.004681583;_utc_var:3.338563536827082e-08;type:32;_utc:491546775.4343481;_sf_var:3.591880115602739e-12
2016-07-30 00:46:15 -0400 timed[62]: need_rtc_s:9170734.868813347;bb_want_rtc_s:inf;type:20;rtc_s:8670744.061904;want_rtc_s:8673234.868813347
2016-07-30 00:46:15 -0400 timed[62]: type:21;rtc_s:8670744.064727001;mach:144094750487
2016-07-30 00:46:15 -0400 timed[62]: t_s:491546775.4751505;adjust:5.14984130859375e-05;didSet:1;mach:144094288667;rtc_s:8670744.0454845;unc_s:0.0001827755480512193;after_mach:144094288687;type:3;settimeofday:0
2016-07-30 00:46:16 -0400 timed[62]: type:21;rtc_s:8670745.088287;mach:144119318201
2016-07-30 00:46:16 -0400 timed[62]: unc_ns:500925;src:13;before_mach:144118686001;after_mach:144118686150;rtc_s:8670745.061945332;type:11;t_ns:491546776491588677
2016-07-30 00:46:16 -0400 timed[62]: t_s:491546776.491599;rateSf:0.9999877993098584;sf_var:3.621272047817164e-12;rtc_s:8670745.061945332;utc_var:3.39137766960668e-08;type:29;utc_sf_cov:2.516771519221459e-10
2016-07-30 00:46:16 -0400 timed[62]: _utc_sf_cov:2.217118776274168e-10;_rateSf:0.9999877901988109;_rtc:8670745.061945332;_utc_var:2.987592259089083e-08;type:32;_utc:491546776.4915977;_sf_var:3.398897083251865e-12
2016-07-30 00:46:16 -0400 timed[62]: need_rtc_s:9170736.419618882;bb_want_rtc_s:inf;type:20;rtc_s:8670745.130874;want_rtc_s:8673236.419618882
2016-07-30 00:46:16 -0400 timed[62]: type:21;rtc_s:8670745.133208999;mach:144120393818
2016-07-30 00:46:16 -0400 timed[62]: t_s:491546776.5229479;adjust:-4.112720489501953e-06;didSet:1;mach:144119435905;rtc_s:8670745.093295958;unc_s:0.0001728911079953641;after_mach:144119435923;type:3;settimeofday:0
2016-07-30 00:47:10 -0400 timed[62]: type:21;rtc_s:8670799.496703999;mach:145425118052
2016-07-30 00:47:10 -0400 timed[62]: unc_ns:601319;src:16;before_mach:145424202317;after_mach:145424202445;rtc_s:8670799.458548374;type:11;t_ns:491546830887411150
2016-07-30 00:47:10 -0400 timed[62]: t_s:491546830.8875366;rateSf:0.9999877901988109;sf_var:4.911122647826687e-12;rtc_s:8670799.458548374;utc_var:6.554552615562564e-08;type:29;utc_sf_cov:4.477302999198301e-10
2016-07-30 00:47:10 -0400 timed[62]: _utc_sf_cov:3.790235730232265e-10;_rateSf:0.9999876587423169;_rtc:8670799.458548374;_utc_var:5.548719736779257e-08;type:32;_utc:491546830.8875173;_sf_var:4.441798725292487e-12
2016-07-30 00:47:10 -0400 timed[62]: need_rtc_s:9170787.680688066;bb_want_rtc_s:inf;type:20;rtc_s:8670799.508865001;want_rtc_s:8673287.680688066
2016-07-30 00:47:10 -0400 timed[62]: type:21;rtc_s:8670799.51065;mach:145425451760
2016-07-30 00:47:10 -0400 timed[62]: t_s:491546830.9283709;adjust:-0.0006570816040039062;didSet:1;mach:145425181819;rtc_s:8670799.499402458;unc_s:0.0002356262861597482;after_mach:145425181830;type:3;settimeofday:0
2016-07-30 00:47:12 -0400 timed[62]: type:21;rtc_s:8670800.605377;mach:145451728084
2016-07-30 00:47:12 -0400 timed[62]: unc_ns:501002;src:13;before_mach:145451004347;after_mach:145451004520;rtc_s:8670800.575221291;type:11;t_ns:491546832004140054
2016-07-30 00:47:12 -0400 timed[62]: t_s:491546832.0041765;rateSf:0.9999876587423169;sf_var:4.472842232366679e-12;rtc_s:8670800.575221291;utc_var:5.633923972665137e-08;type:29;utc_sf_cov:3.840009420813736e-10
2016-07-30 00:47:12 -0400 timed[62]: _utc_sf_cov:3.136093394949686e-10;_rateSf:0.9999876132402437;_rtc:8670800.575221291;_utc_var:4.601163648858887e-08;type:32;_utc:491546832.0041698;_sf_var:3.993062209793702e-12
2016-07-30 00:47:12 -0400 timed[62]: type:21;rtc_s:8670800.611785;mach:145451881513
2016-07-30 00:47:12 -0400 timed[62]: need_rtc_s:9170789.850059697;bb_want_rtc_s:inf;type:20;rtc_s:8670800.623060999;want_rtc_s:8673289.850059697
2016-07-30 00:47:12 -0400 timed[62]: t_s:491546832.0385267;adjust:-0.0001253485679626465;didSet:1;mach:145451828558;rtc_s:8670800.609578542;unc_s:0.0002145564072048076;after_mach:145451828583;type:3;settimeofday:0
2016-07-30 00:47:13 -0400 timed[62]: type:21;rtc_s:8670801.685485;mach:145477650450
2016-07-30 00:47:13 -0400 timed[62]: unc_ns:500504;src:13;before_mach:145477523305;after_mach:145477523427;rtc_s:8670801.680187291;type:11;t_ns:491546833109080505
2016-07-30 00:47:13 -0400 timed[62]: t_s:491546833.1091222;rateSf:0.9999876132402437;sf_var:4.023780264585987e-12;rtc_s:8670801.680187291;utc_var:4.670957963374669e-08;type:29;utc_sf_cov:3.180385086746289e-10
2016-07-30 00:47:13 -0400 timed[62]: _utc_sf_cov:2.680563220258498e-10;_rateSf:0.9999875686575225;_rtc:8670801.680187291;_utc_var:3.936881156993839e-08;type:32;_utc:491546833.1091156;_sf_var:3.683459066520871e-12
2016-07-30 00:47:13 -0400 timed[62]: type:21;rtc_s:8670801.688934;mach:145477732596
2016-07-30 00:47:13 -0400 timed[62]: need_rtc_s:9170791.759399574;bb_want_rtc_s:inf;type:20;rtc_s:8670801.699142;want_rtc_s:8673291.759399574
2016-07-30 00:47:13 -0400 timed[62]: t_s:491546833.1159739;adjust:8.940696716308594e-07;didSet:0;mach:145477687276;rtc_s:8670801.687045667;unc_s:0.0001984282215194003;after_mach:145477687289;type:3;settimeofday:0
2016-07-30 00:47:13 -0400 timed[62]: type:21;rtc_s:8670801.731719;mach:145478757914
2016-07-30 00:47:13 -0400 timed[62]: t_s:491546833.1601442;adjust:6.312131881713867e-05;didSet:1;mach:145478745853;rtc_s:8670801.731216459;unc_s:0.0001984875678709737;after_mach:145478745865;type:3;settimeofday:0
2016-07-30 00:47:14 -0400 timed[62]: type:21;rtc_s:8670802.735198;mach:145502844847
2016-07-30 00:47:14 -0400 timed[62]: unc_ns:500940;src:13;before_mach:145502195275;after_mach:145502195392;rtc_s:8670802.7081325;type:11;t_ns:491546834137066113
2016-07-30 00:47:14 -0400 timed[62]: t_s:491546834.137048;rateSf:0.9999875686575225;sf_var:3.71203594333926e-12;rtc_s:8670802.7081325;utc_var:3.992380826516596e-08;type:29;utc_sf_cov:2.718574038587236e-10
2016-07-30 00:47:14 -0400 timed[62]: _utc_sf_cov:2.345425575356101e-10;_rateSf:0.9999875855932315;_rtc:8670802.7081325;_utc_var:3.444391053605244e-08;type:32;_utc:491546834.1370505;_sf_var:3.457944019557532e-12
2016-07-30 00:47:14 -0400 timed[62]: type:21;rtc_s:8670802.738647001;mach:145502927663
2016-07-30 00:47:14 -0400 timed[62]: need_rtc_s:9170793.428597149;bb_want_rtc_s:inf;type:20;rtc_s:8670802.751327001;want_rtc_s:8673293.428597149
2016-07-30 00:47:14 -0400 timed[62]: t_s:491546834.1660771;adjust:-0.0001548528671264648;didSet:1;mach:145502891963;rtc_s:8670802.737159502;unc_s:0.0001856312635014569;after_mach:145502891981;type:3;settimeofday:0
2016-07-30 01:02:53 -0400 timed[62]: type:21;rtc_s:8671741.903594;mach:168042913476
2016-07-30 01:02:53 -0400 timed[62]: unc_ns:501692;src:14;before_mach:168042303068;after_mach:168042303185;rtc_s:8671741.878160333;type:11;t_ns:491547773295483314
2016-07-30 01:02:53 -0400 timed[62]: t_s:491547773.2954191;rateSf:0.9999875855932315;sf_var:2.956687079332694e-11;rtc_s:8671741.878160333;utc_var:1.120141627370657e-05;type:29;utc_sf_cov:1.574250068104093e-08
2016-07-30 01:02:53 -0400 timed[62]: _utc_sf_cov:3.459598798000078e-10;_rateSf:0.9999876738292753;_rtc:8671741.878160333;_utc_var:2.461642343969164e-07;type:32;_utc:491547773.2954819;_sf_var:7.928530772149811e-12
2016-07-30 01:02:53 -0400 timed[62]: type:21;rtc_s:8671741.906432999;mach:168042981046
2016-07-30 01:02:53 -0400 timed[62]: need_rtc_s:9171717.070689967;bb_want_rtc_s:inf;type:20;rtc_s:8671741.917129001;want_rtc_s:8674217.070689967
2016-07-30 01:02:53 -0400 timed[62]: t_s:491547773.3229969;adjust:-0.01270210742950439;didSet:1;mach:168042962872;rtc_s:8671741.90567575;unc_s:0.0004961719530274458;after_mach:168042962886;type:3;settimeofday:0
2016-07-30 01:02:54 -0400 timed[62]: type:21;rtc_s:8671742.952117;mach:168068076108
2016-07-30 01:02:54 -0400 timed[62]: unc_ns:500894;src:14;before_mach:168067481308;after_mach:168067481408;rtc_s:8671742.927333666;type:11;t_ns:491547774344563194
2016-07-30 01:02:54 -0400 timed[62]: t_s:491547774.3446423;rateSf:0.9999876738292753;sf_var:7.957697790800283e-12;rtc_s:8671742.927333666;utc_var:2.468989163057555e-07;type:29;utc_sf_cov:3.542935834831234e-10
2016-07-30 01:02:54 -0400 timed[62]: _utc_sf_cov:1.785690178131572e-10;_rateSf:0.9999876175349884;_rtc:8671742.927333666;_utc_var:1.244405742559879e-07;type:32;_utc:491547774.3446031;_sf_var:7.705537566653618e-12
2016-07-30 01:02:54 -0400 timed[62]: type:21;rtc_s:8671742.954314999;mach:168068129664
2016-07-30 01:02:54 -0400 timed[62]: need_rtc_s:9171725.289265867;bb_want_rtc_s:inf;type:20;rtc_s:8671742.964492001;want_rtc_s:8674225.289265867
2016-07-30 01:02:54 -0400 timed[62]: t_s:491547774.3705332;adjust:-0.008652746677398682;didSet:1;mach:168068104444;rtc_s:8671742.953264166;unc_s:0.0003527775138497614;after_mach:168068104495;type:3;settimeofday:0
2016-07-30 01:02:55 -0400 timed[62]: type:21;rtc_s:8671743.642059;mach:168084634625
2016-07-30 01:02:55 -0400 timed[62]: type:9;src:10;rtc_s:8671743.640914001
2016-07-30 01:02:55 -0400 timed[62]: type:21;rtc_s:8671744.074981;mach:168095027038
2016-07-30 01:02:55 -0400 timed[62]: unc_ns:500911;src:13;before_mach:168094412498;after_mach:168094412579;rtc_s:8671744.049375167;type:11;t_ns:491547775466682099
2016-07-30 01:02:55 -0400 timed[62]: t_s:491547775.4666307;rateSf:0.9999876175349884;sf_var:7.736730320384333e-12;rtc_s:8671744.049375167;utc_var:1.248510121394772e-07;type:29;utc_sf_cov:1.872324505333747e-10
2016-07-30 01:02:55 -0400 timed[62]: _utc_sf_cov:1.250225908377676e-10;_rateSf:0.9999876431358466;_rtc:8671744.049375167;_utc_var:8.336801105753102e-08;type:32;_utc:491547775.4666477;_sf_var:7.643437488563844e-12
2016-07-30 01:02:55 -0400 timed[62]: type:21;rtc_s:8671744.077606;mach:168095087996
2016-07-30 01:02:55 -0400 timed[62]: need_rtc_s:9171729.61261557;bb_want_rtc_s:inf;type:20;rtc_s:8671744.088270999;want_rtc_s:8674229.61261557
2016-07-30 01:02:55 -0400 timed[62]: t_s:491547775.4941037;adjust:-0.004143297672271729;didSet:1;mach:168095069409;rtc_s:8671744.076831542;unc_s:0.0002887507598025679;after_mach:168095069476;type:3;settimeofday:0
2016-07-30 01:02:56 -0400 timed[62]: type:21;rtc_s:8671745.193236999;mach:168121864530
2016-07-30 01:02:56 -0400 timed[62]: unc_ns:500901;src:13;before_mach:168121261197;after_mach:168121261342;rtc_s:8671745.168098124;type:11;t_ns:491547776585367233
2016-07-30 01:02:56 -0400 timed[62]: t_s:491547776.5853569;rateSf:0.9999876431358466;sf_var:7.674537986758092e-12;rtc_s:8671745.168098124;utc_var:8.365732139279053e-08;type:29;utc_sf_cov:1.335908762444183e-10
2016-07-30 01:02:56 -0400 timed[62]: _utc_sf_cov:1.001862319189041e-10;_rateSf:0.9999876472771043;_rtc:8671745.168098124;_utc_var:6.273865430329189e-08;type:32;_utc:491547776.5853595;_sf_var:7.621194709304517e-12
2016-07-30 01:02:56 -0400 timed[62]: type:21;rtc_s:8671745.195892001;mach:168121928259
2016-07-30 01:02:56 -0400 timed[62]: need_rtc_s:9171732.644255433;bb_want_rtc_s:inf;type:20;rtc_s:8671745.205642;want_rtc_s:8674232.644255433
2016-07-30 01:02:56 -0400 timed[62]: t_s:491547776.6127838;adjust:1.287460327148438e-05;didSet:1;mach:168121919399;rtc_s:8671745.195522834;unc_s:0.0002504946712953378;after_mach:168121919411;type:3;settimeofday:0
2016-07-30 01:02:57 -0400 timed[62]: type:21;rtc_s:8671746.293243;mach:168148264638
2016-07-30 01:02:57 -0400 timed[62]: unc_ns:500902;src:13;before_mach:168147660724;after_mach:168147660836;rtc_s:8671746.268079918;type:11;t_ns:491547777685333494
2016-07-30 01:02:57 -0400 timed[62]: t_s:491547777.6853277;rateSf:0.9999876472771043;sf_var:7.651774203181436e-12;rtc_s:8671746.268079918;utc_var:6.296829403922993e-08;type:29;utc_sf_cov:1.08586225791957e-10
2016-07-30 01:02:57 -0400 timed[62]: _utc_sf_cov:8.680185788663671e-11;_rateSf:0.999987649277311;_rtc:8671746.268079918;_utc_var:5.033571128099787e-08;type:32;_utc:491547777.6853288;_sf_var:7.614207959277345e-12
2016-07-30 01:02:57 -0400 timed[62]: type:21;rtc_s:8671746.295042999;mach:168148306585
2016-07-30 01:02:57 -0400 timed[62]: need_rtc_s:9171735.05026916;bb_want_rtc_s:inf;type:20;rtc_s:8671746.306564;want_rtc_s:8674235.05026916
2016-07-30 01:02:57 -0400 timed[62]: t_s:491547777.7119997;adjust:4.273653030395508e-05;didSet:1;mach:168148299581;rtc_s:8671746.294751165;unc_s:0.0002243700989905885;after_mach:168148299594;type:3;settimeofday:0
2016-07-30 01:02:58 -0400 timed[62]: type:21;rtc_s:8671747.323364001;mach:168172987694
2016-07-30 01:02:58 -0400 timed[62]: unc_ns:500904;src:13;before_mach:168172380534;after_mach:168172380616;rtc_s:8671747.298065668;type:11;t_ns:491547778715311417
2016-07-30 01:02:58 -0400 timed[62]: t_s:491547778.7153019;rateSf:0.999987649277311;sf_var:7.64284156312996e-12;rtc_s:8671747.298065668;utc_var:5.052260844963627e-08;type:29;utc_sf_cov:9.465912968491614e-11
2016-07-30 01:02:58 -0400 timed[62]: _utc_sf_cov:7.879325162178421e-11;_rateSf:0.9999876522721841;_rtc:8671747.298065668;_utc_var:4.205448131006232e-08;type:32;_utc:491547778.7153035;_sf_var:7.613115263256901e-12
2016-07-30 01:02:58 -0400 timed[62]: type:21;rtc_s:8671747.32669;mach:168173067662
2016-07-30 01:02:58 -0400 timed[62]: need_rtc_s:9171737.044471016;bb_want_rtc_s:inf;type:20;rtc_s:8671747.335341999;want_rtc_s:8674237.044471016
2016-07-30 01:02:58 -0400 timed[62]: t_s:491547778.7426522;adjust:-7.575750350952148e-05;didSet:1;mach:168173037056;rtc_s:8671747.325414749;unc_s:0.0002050900203857352;after_mach:168173037073;type:3;settimeofday:0
2016-07-30 01:06:31 -0400 timed[62]: type:21;rtc_s:8671960.077453;mach:173279093296
2016-07-30 01:06:31 -0400 timed[62]: t_s:491547991.4914777;adjust:-0.002932310104370117;didSet:1;mach:173279079236;rtc_s:8671960.076867167;unc_s:0.0007138249787937713;after_mach:173279079257;type:3;settimeofday:0
2016-07-30 01:32:53 -0400 timed[62]: type:21;rtc_s:8673541.961363999;mach:211244458672
2016-07-30 01:32:53 -0400 timed[62]: unc_ns:501756;src:14;before_mach:211243819173;after_mach:211243819350;rtc_s:8673541.934718208;type:11;t_ns:491549573331502258
2016-07-30 01:32:53 -0400 timed[62]: t_s:491549573.3297963;rateSf:0.9999876522721841;sf_var:5.750401420388045e-11;rtc_s:8673541.934718208;utc_var:7.840613359068473e-05;type:29;utc_sf_cov:5.85095868765939e-08
2016-07-30 01:32:53 -0400 timed[62]: _utc_sf_cov:1.872713985904528e-10;_rateSf:0.999988921237136;_rtc:8673541.934718208;_utc_var:2.509541953623205e-07;type:32;_utc:491549573.3314968;_sf_var:1.398172292347249e-11
2016-07-30 01:32:53 -0400 timed[62]: need_rtc_s:9173516.887053877;bb_want_rtc_s:inf;type:20;rtc_s:8673541.988508999;want_rtc_s:8676016.887053877
2016-07-30 01:32:53 -0400 timed[62]: type:21;rtc_s:8673541.991576999;mach:211245180952
2016-07-30 01:32:53 -0400 timed[62]: t_s:491549573.3699768;adjust:-0.0240282416343689;didSet:1;mach:211244739871;rtc_s:8673541.973198624;unc_s:0.0005009756526271376;after_mach:211244739897;type:3;settimeofday:0
2016-07-30 01:32:54 -0400 timed[62]: type:21;rtc_s:8673543.085571;mach:211271439343
2016-07-30 01:32:54 -0400 timed[62]: unc_ns:500944;src:14;before_mach:211270784460;after_mach:211270784543;rtc_s:8673543.058284208;type:11;t_ns:491549574455034641
2016-07-30 01:32:54 -0400 timed[62]: t_s:491549574.4550503;rateSf:0.999988921237136;sf_var:1.401295805826668e-11;rtc_s:8673543.058284208;utc_var:2.51392682593241e-07;type:29;utc_sf_cov:2.029983344534921e-10
2016-07-30 01:32:54 -0400 timed[62]: _utc_sf_cov:1.014088383956011e-10;_rateSf:0.9999889148782622;_rtc:8673543.058284208;_utc_var:1.255844782745017e-07;type:32;_utc:491549574.4550425;_sf_var:1.393092504805736e-11
2016-07-30 01:32:54 -0400 timed[62]: type:21;rtc_s:8673543.089569001;mach:211271535310
2016-07-30 01:32:54 -0400 timed[62]: need_rtc_s:9173525.339334022;bb_want_rtc_s:inf;type:20;rtc_s:8673543.101699;want_rtc_s:8676025.339334022
2016-07-30 01:32:54 -0400 timed[62]: t_s:491549574.4841324;adjust:-0.02003759145736694;didSet:1;mach:211271482641;rtc_s:8673543.087374458;unc_s:0.0003543901135100103;after_mach:211271482771;type:3;settimeofday:0
2016-07-30 01:32:55 -0400 timed[62]: type:21;rtc_s:8673543.656127;mach:211285130331
2016-07-30 01:32:55 -0400 timed[62]: type:9;src:10;rtc_s:8673543.653853999
2016-07-30 01:32:55 -0400 timed[62]: type:21;rtc_s:8673544.194427;mach:211298052115
2016-07-30 01:32:55 -0400 timed[62]: unc_ns:500947;src:13;before_mach:211297394075;after_mach:211297394250;rtc_s:8673544.167008666;type:11;t_ns:491549575563754769
2016-07-30 01:32:55 -0400 timed[62]: t_s:491549575.5637547;rateSf:0.9999889148782622;sf_var:1.396174758799374e-11;rtc_s:8673544.167008666;utc_var:1.258264846902206e-07;type:29;utc_sf_cov:1.168714825728856e-10
2016-07-30 01:32:55 -0400 timed[62]: _utc_sf_cov:7.784145771809972e-11;_rateSf:0.9999889149152397;_rtc:8673544.167008666;_utc_var:8.380587609747045e-08;type:32;_utc:491549575.5637547;_sf_var:1.392549530866744e-11
2016-07-30 01:32:55 -0400 timed[62]: type:21;rtc_s:8673544.202238999;mach:211298238699
2016-07-30 01:32:55 -0400 timed[62]: need_rtc_s:9173529.692386383;bb_want_rtc_s:inf;type:20;rtc_s:8673544.211805999;want_rtc_s:8676029.692386383
2016-07-30 01:32:55 -0400 timed[62]: t_s:491549575.5945342;adjust:-0.01558178663253784;didSet:1;mach:211298131887;rtc_s:8673544.197788499;unc_s:0.0002895035561751924;after_mach:211298132011;type:3;settimeofday:0
2016-07-30 01:32:56 -0400 timed[62]: type:21;rtc_s:8673545.230346;mach:211322914177
2016-07-30 01:32:56 -0400 timed[62]: unc_ns:500929;src:13;before_mach:211322277958;after_mach:211322278120;rtc_s:8673545.203836875;type:11;t_ns:491549576600566554
2016-07-30 01:32:56 -0400 timed[62]: t_s:491549576.6005715;rateSf:0.9999889149152397;sf_var:1.395431913286971e-11;rtc_s:8673545.203836875;utc_var:8.398227294925847e-08;type:29;utc_sf_cov:9.229474675145864e-11
2016-07-30 01:32:56 -0400 timed[62]: _utc_sf_cov:6.915100978604547e-11;_rateSf:0.9999889135683259;_rtc:8673545.203836875;_utc_var:6.292296347274752e-08;type:32;_utc:491549576.6005702;_sf_var:1.39288846540554e-11
2016-07-30 01:32:56 -0400 timed[62]: need_rtc_s:9173532.661611846;bb_want_rtc_s:inf;type:20;rtc_s:8673545.251173999;want_rtc_s:8676032.661611846
2016-07-30 01:32:56 -0400 timed[62]: type:21;rtc_s:8673545.253753001;mach:211323473604
2016-07-30 01:32:56 -0400 timed[62]: t_s:491549576.6312513;adjust:-0.01137369871139526;didSet:1;mach:211323011971;rtc_s:8673545.234518293;unc_s:0.0002508573293714851;after_mach:211323012066;type:3;settimeofday:0
2016-07-30 01:32:57 -0400 timed[62]: type:21;rtc_s:8673546.272399001;mach:211347922728
2016-07-30 01:32:57 -0400 timed[62]: unc_ns:500934;src:13;before_mach:211347280042;after_mach:211347280192;rtc_s:8673546.245620418;type:11;t_ns:491549577642303167
2016-07-30 01:32:57 -0400 timed[62]: t_s:491549577.6423422;rateSf:0.9999889135683259;sf_var:1.395784623655924e-11;rtc_s:8673546.245620418;utc_var:6.308217191711297e-08;type:29;utc_sf_cov:8.367697844523484e-11
2016-07-30 01:32:57 -0400 timed[62]: _utc_sf_cov:6.6867330661516e-11;_rateSf:0.9999889031649718;_rtc:8673546.245620418;_utc_var:5.040976056740493e-08;type:32;_utc:491549577.6423343;_sf_var:1.393554864450553e-11
2016-07-30 01:32:57 -0400 timed[62]: type:21;rtc_s:8673546.276427999;mach:211348019272
2016-07-30 01:32:57 -0400 timed[62]: need_rtc_s:9173535.019561386;bb_want_rtc_s:inf;type:20;rtc_s:8673546.291060999;want_rtc_s:8676035.019561386
2016-07-30 01:32:57 -0400 timed[62]: t_s:491549577.6710389;adjust:-0.007335066795349121;didSet:1;mach:211347968808;rtc_s:8673546.274325332;unc_s:0.0002245343320519679;after_mach:211347968930;type:3;settimeofday:0
2016-07-30 01:32:58 -0400 timed[62]: type:21;rtc_s:8673547.356170001;mach:211373931669
2016-07-30 01:32:58 -0400 timed[62]: unc_ns:500944;src:13;before_mach:211373276701;after_mach:211373276870;rtc_s:8673547.328879667;type:11;t_ns:491549578725481158
2016-07-30 01:32:58 -0400 timed[62]: t_s:491549578.7255816;rateSf:0.9999889031649718;sf_var:1.396566325163068e-11;rtc_s:8673547.328879667;utc_var:5.057099433347278e-08;type:29;utc_sf_cov:8.197945358539465e-11
2016-07-30 01:32:58 -0400 timed[62]: _utc_sf_cov:6.822969256086518e-11;_rateSf:0.9999888758579921;_rtc:8673547.328879667;_utc_var:4.208912410322322e-08;type:32;_utc:491549578.7255647;_sf_var:1.394337383631312e-11
2016-07-30 01:32:58 -0400 timed[62]: need_rtc_s:9173537.071062636;bb_want_rtc_s:inf;type:20;rtc_s:8673547.372633999;want_rtc_s:8676037.071062636
2016-07-30 01:32:58 -0400 timed[62]: type:21;rtc_s:8673547.375091;mach:211374386421
2016-07-30 01:32:58 -0400 timed[62]: t_s:491549578.7558025;adjust:-0.003051519393920898;didSet:1;mach:211374003063;rtc_s:8673547.35911775;unc_s:0.0002051700807196077;after_mach:211374003160;type:3;settimeofday:0
2016-07-30 01:32:59 -0400 timed[62]: type:21;rtc_s:8673548.447601;mach:211400129024
2016-07-30 01:32:59 -0400 timed[62]: unc_ns:500956;src:13;before_mach:211399460196;after_mach:211399460366;rtc_s:8673548.419733167;type:11;t_ns:491549579816443126
2016-07-30 01:32:59 -0400 timed[62]: t_s:491549579.8164061;rateSf:0.9999888758579921;sf_var:1.397369956359223e-11;rtc_s:8673548.419733167;utc_var:4.225458540585205e-08;type:29;utc_sf_cov:8.345641116439889e-11
2016-07-30 01:32:59 -0400 timed[62]: _utc_sf_cov:7.142955763598516e-11;_rateSf:0.9999888864103295;_rtc:8673548.419733167;_utc_var:3.616530235989217e-08;type:32;_utc:491549579.8164114;_sf_var:1.394994550179182e-11
2016-07-30 01:32:59 -0400 timed[62]: type:21;rtc_s:8673548.451416001;mach:211400218312
2016-07-30 01:32:59 -0400 timed[62]: need_rtc_s:9173538.911144633;bb_want_rtc_s:inf;type:20;rtc_s:8673548.465102;want_rtc_s:8676038.911144633
2016-07-30 01:32:59 -0400 timed[62]: t_s:491549579.8466359;adjust:0.0002718567848205566;didSet:1;mach:211400183318;rtc_s:8673548.449957917;unc_s:0.0001901877856310201;after_mach:211400183338;type:3;settimeofday:0
2016-07-30 01:33:00 -0400 timed[62]: type:21;rtc_s:8673549.477538999;mach:211424847267
2016-07-30 01:33:00 -0400 timed[62]: unc_ns:500937;src:13;before_mach:211424200749;after_mach:211424200925;rtc_s:8673549.450600749;type:11;t_ns:491549580847283686
2016-07-30 01:33:00 -0400 timed[62]: t_s:491549580.8472676;rateSf:0.9999888864103295;sf_var:1.397860362057662e-11;rtc_s:8673549.450600749;utc_var:3.632740578148687e-08;type:29;utc_sf_cov:8.582487558986936e-11
2016-07-30 01:33:00 -0400 timed[62]: _utc_sf_cov:7.497153918031315e-11;_rateSf:0.9999888912184213;_rtc:8673549.450600749;_utc_var:3.173347478976547e-08;type:32;_utc:491549580.8472696;_sf_var:1.395296220118411e-11
2016-07-30 01:33:00 -0400 timed[62]: need_rtc_s:9173540.543654753;bb_want_rtc_s:inf;type:20;rtc_s:8673549.493514;want_rtc_s:8676040.543654753
2016-07-30 01:33:00 -0400 timed[62]: type:21;rtc_s:8673549.495634999;mach:211425278847
2016-07-30 01:33:00 -0400 timed[62]: t_s:491549580.878366;adjust:2.199411392211914e-05;didSet:1;mach:211424944346;rtc_s:8673549.481697457;unc_s:0.0001781566933101794;after_mach:211424944367;type:3;settimeofday:0
2016-07-30 01:44:00 -0400 timed[62]: type:21;rtc_s:8674209.383544;mach:227262669269
2016-07-30 01:44:00 -0400 timed[62]: unc_ns:533947;src:14;before_mach:227262015103;after_mach:227262015280;rtc_s:8674209.356287083;type:11;t_ns:491550240745797700
2016-07-30 01:44:00 -0400 timed[62]: t_s:491550240.7456252;rateSf:0.9999888912184213;sf_var:3.229834028126306e-11;rtc_s:8674209.356287083;utc_var:8.869842818029746e-06;type:29;utc_sf_cov:1.533572029343549e-08
2016-07-30 01:44:00 -0400 timed[62]: _utc_sf_cov:4.775788560242182e-10;_rateSf:0.9999891801714129;_rtc:8674209.356287083;_utc_var:2.762210907017214e-07;type:32;_utc:491550240.7457923;_sf_var:6.609012314676021e-12
2016-07-30 01:44:00 -0400 timed[62]: need_rtc_s:9174183.077917142;bb_want_rtc_s:inf;type:20;rtc_s:8674209.403152;want_rtc_s:8676683.077917142
2016-07-30 01:44:00 -0400 timed[62]: type:21;rtc_s:8674209.405639;mach:227263198158
2016-07-30 01:44:00 -0400 timed[62]: t_s:491550240.7776117;adjust:-0.01046329736709595;didSet:1;mach:227262777385;rtc_s:8674209.388106791;unc_s:0.0005256050583841363;after_mach:227262777405;type:3;settimeofday:0
2016-07-30 01:44:01 -0400 timed[62]: type:21;rtc_s:8674210.492858;mach:227289294474
2016-07-30 01:44:01 -0400 timed[62]: unc_ns:501129;src:14;before_mach:227288653449;after_mach:227288653619;rtc_s:8674210.466148624;type:11;t_ns:491550241855710876
2016-07-30 01:44:01 -0400 timed[62]: t_s:491550241.8556418;rateSf:0.9999891801714129;sf_var:6.639866465530817e-12;rtc_s:8674210.466148624;utc_var:2.772893371138035e-07;type:29;utc_sf_cov:4.84931066537553e-10
2016-07-30 01:44:01 -0400 timed[62]: _utc_sf_cov:2.304625283261304e-10;_rateSf:0.9999892435130945;_rtc:8674210.466148624;_utc_var:1.317812079259209e-07;type:32;_utc:491550241.8556781;_sf_var:6.194845027203846e-12
2016-07-30 01:44:01 -0400 timed[62]: need_rtc_s:9174192.315307878;bb_want_rtc_s:inf;type:20;rtc_s:8674210.513350999;want_rtc_s:8676692.315307878
2016-07-30 01:44:01 -0400 timed[62]: type:21;rtc_s:8674210.515838001;mach:227289843390
2016-07-30 01:44:01 -0400 timed[62]: t_s:491550241.8868777;adjust:-0.006341278553009033;didSet:1;mach:227289399645;rtc_s:8674210.497348625;unc_s:0.0003630405539150091;after_mach:227289399739;type:3;settimeofday:0
2016-07-30 01:44:03 -0400 timed[62]: type:21;rtc_s:8674211.618377;mach:227316307435
2016-07-30 01:44:03 -0400 timed[62]: unc_ns:500947;src:13;before_mach:227315649026;after_mach:227315649202;rtc_s:8674211.590943292;type:11;t_ns:491550242980508885
2016-07-30 01:44:03 -0400 timed[62]: t_s:491550242.9804606;rateSf:0.9999892435130945;sf_var:6.226114318962756e-12;rtc_s:8674211.590943292;utc_var:1.323075046477111e-07;type:29;utc_sf_cov:2.3744804274557e-10
2016-07-30 01:44:03 -0400 timed[62]: _utc_sf_cov:1.554763274548328e-10;_rateSf:0.9999892733881121;_rtc:8674211.590943292;_utc_var:8.663236251385407e-08;type:32;_utc:491550242.9804773;_sf_var:6.079002305867893e-12
2016-07-30 01:44:03 -0400 timed[62]: type:21;rtc_s:8674211.622467;mach:227316406677
2016-07-30 01:44:03 -0400 timed[62]: need_rtc_s:9174196.874255272;bb_want_rtc_s:inf;type:20;rtc_s:8674211.635635;want_rtc_s:8676696.874255272
2016-07-30 01:44:03 -0400 timed[62]: t_s:491550243.0097237;adjust:-0.002032279968261719;didSet:1;mach:227316352030;rtc_s:8674211.620190041;unc_s:0.0002943515357658966;after_mach:227316352155;type:3;settimeofday:0
2016-07-30 01:44:04 -0400 timed[62]: type:21;rtc_s:8674212.719207;mach:227342726848
2016-07-30 01:44:04 -0400 timed[62]: unc_ns:500525;src:13;before_mach:227342575351;after_mach:227342575490;rtc_s:8674212.712894624;type:11;t_ns:491550244102419036
2016-07-30 01:44:04 -0400 timed[62]: t_s:491550244.1024166;rateSf:0.9999892733881121;sf_var:6.110192552906273e-12;rtc_s:8674212.712894624;utc_var:8.698890144134633e-08;type:29;utc_sf_cov:1.623141691606621e-10
2016-07-30 01:44:04 -0400 timed[62]: _utc_sf_cov:1.204803089607073e-10;_rateSf:0.9999892745633561;_rtc:8674212.712894624;_utc_var:6.456891456858673e-08;type:32;_utc:491550244.1024172;_sf_var:6.03213399430133e-12
2016-07-30 01:44:04 -0400 timed[62]: type:21;rtc_s:8674212.722960999;mach:227342817017
2016-07-30 01:44:04 -0400 timed[62]: need_rtc_s:9174200.007687565;bb_want_rtc_s:inf;type:20;rtc_s:8674212.734511999;want_rtc_s:8676700.007687565
2016-07-30 01:44:04 -0400 timed[62]: t_s:491550244.1104332;adjust:0.000375211238861084;didSet:1;mach:227342767810;rtc_s:8674212.720910707;unc_s:0.0002541111196251553;after_mach:227342767829;type:3;settimeofday:0
2016-07-30 01:44:05 -0400 timed[62]: type:21;rtc_s:8674213.764373001;mach:227367810916
2016-07-30 01:44:05 -0400 timed[62]: unc_ns:500940;src:13;before_mach:227367161625;after_mach:227367161768;rtc_s:8674213.737319209;type:11;t_ns:491550245126831851
2016-07-30 01:44:05 -0400 timed[62]: t_s:491550245.1268308;rateSf:0.9999892745633561;sf_var:6.060612997752722e-12;rtc_s:8674213.737319209;utc_var:6.48221009091512e-08;type:29;utc_sf_cov:1.266743626176051e-10
2016-07-30 01:44:05 -0400 timed[62]: _utc_sf_cov:1.006697428765022e-10;_rateSf:0.9999892749698522;_rtc:8674213.737319209;_utc_var:5.151495611576892e-08;type:32;_utc:491550245.126831;_sf_var:6.009795165800096e-12
2016-07-30 01:44:05 -0400 timed[62]: type:21;rtc_s:8674213.768859001;mach:227367919022
2016-07-30 01:44:05 -0400 timed[62]: need_rtc_s:9174202.388865998;bb_want_rtc_s:inf;type:20;rtc_s:8674213.781570001;want_rtc_s:8676702.388865998
2016-07-30 01:44:05 -0400 timed[62]: t_s:491550245.1560901;adjust:-2.884864807128906e-05;didSet:1;mach:227367864294;rtc_s:8674213.766578667;unc_s:0.0002269849015230258;after_mach:227367864314;type:3;settimeofday:0
2016-07-30 01:44:06 -0400 timed[62]: type:21;rtc_s:8674214.781646;mach:227392222872
2016-07-30 01:44:06 -0400 timed[62]: unc_ns:500481;src:13;before_mach:227392123936;after_mach:227392124022;rtc_s:8674214.777523667;type:11;t_ns:491550246166908643
2016-07-30 01:44:06 -0400 timed[62]: t_s:491550246.1670243;rateSf:0.9999892749698522;sf_var:6.038712849730791e-12;rtc_s:8674214.777523667;utc_var:5.17309035268002e-08;type:29;utc_sf_cov:1.069361987511347e-10
2016-07-30 01:44:06 -0400 timed[62]: _utc_sf_cov:8.863151353652645e-11;_rateSf:0.99998923403268;_rtc:8674214.777523667;_utc_var:4.287592349212512e-08;type:32;_utc:491550246.1670045;_sf_var:6.000874085612314e-12
2016-07-30 01:44:06 -0400 timed[62]: type:21;rtc_s:8674214.783873999;mach:227392277228
2016-07-30 01:44:06 -0400 timed[62]: need_rtc_s:9174204.424272548;bb_want_rtc_s:inf;type:20;rtc_s:8674214.792831;want_rtc_s:8676704.424272548
2016-07-30 01:44:06 -0400 timed[62]: t_s:491550246.1720148;adjust:-0.000169217586517334;didSet:0;mach:227392245067;rtc_s:8674214.782533959;unc_s:0.0002070693194267155;after_mach:227392250567;type:3;settimeofday:0
2016-07-30 01:44:06 -0400 timed[62]: type:21;rtc_s:8674214.811736999;mach:227392945321
2016-07-30 01:44:06 -0400 timed[62]: t_s:491550246.2009078;adjust:8.380413055419922e-05;didSet:1;mach:227392937889;rtc_s:8674214.811427332;unc_s:0.0002070833538550113;after_mach:227392937895;type:3;settimeofday:0
2016-07-30 01:44:07 -0400 timed[62]: type:21;rtc_s:8674215.905029001;mach:227419185364
2016-07-30 01:44:07 -0400 timed[62]: unc_ns:500884;src:13;before_mach:227418602901;after_mach:227418603014;rtc_s:8674215.880759709;type:11;t_ns:491550247270182400
2016-07-30 01:44:07 -0400 timed[62]: t_s:491550247.2702287;rateSf:0.99998923403268;sf_var:6.031544047578888e-12;rtc_s:8674215.880759709;utc_var:4.307880273806879e-08;type:29;utc_sf_cov:9.526881221477991e-11
2016-07-30 01:44:07 -0400 timed[62]: _utc_sf_cov:8.130769188230393e-11;_rateSf:0.9999892190234723;_rtc:8674215.880759709;_utc_var:3.676584118408955e-08;type:32;_utc:491550247.2702219;_sf_var:6.000669017013398e-12
2016-07-30 01:44:07 -0400 timed[62]: type:21;rtc_s:8674215.907348;mach:227419240852
2016-07-30 01:44:07 -0400 timed[62]: need_rtc_s:9174206.293549323;bb_want_rtc_s:inf;type:20;rtc_s:8674215.91777;want_rtc_s:8676706.293549323
2016-07-30 01:44:07 -0400 timed[62]: t_s:491550247.2956067;adjust:-5.429983139038086e-05;didSet:1;mach:227419211974;rtc_s:8674215.906144749;unc_s:0.0001917568169609836;after_mach:227419211982;type:3;settimeofday:0
2016-07-30 01:44:08 -0400 timed[62]: type:21;rtc_s:8674216.949278999;mach:227444249419
2016-07-30 01:44:08 -0400 timed[62]: unc_ns:500931;src:13;before_mach:227443610521;after_mach:227443610702;rtc_s:8674216.92265825;type:11;t_ns:491550248312153646
2016-07-30 01:44:08 -0400 timed[62]: t_s:491550248.3121092;rateSf:0.9999892190234723;sf_var:6.029633796457437e-12;rtc_s:8674216.92265825;utc_var:3.694179443780657e-08;type:29;utc_sf_cov:8.757486935779284e-11
2016-07-30 01:44:08 -0400 timed[62]: _utc_sf_cov:7.633671670519644e-11;_rateSf:0.9999892325502973;_rtc:8674216.92265825;_utc_var:3.22011933019173e-08;type:32;_utc:491550248.3121149;_sf_var:6.002992433595896e-12
2016-07-30 01:44:08 -0400 timed[62]: type:21;rtc_s:8674216.955505;mach:227444397465
2016-07-30 01:44:08 -0400 timed[62]: need_rtc_s:9174207.950312778;bb_want_rtc_s:inf;type:20;rtc_s:8674216.971755;want_rtc_s:8676707.950312778
2016-07-30 01:44:08 -0400 timed[62]: t_s:491550248.3427211;adjust:-4.190206527709961e-05;didSet:1;mach:227444343701;rtc_s:8674216.953264834;unc_s:0.0001794629949048294;after_mach:227444343719;type:3;settimeofday:0
2016-07-30 01:44:09 -0400 timed[62]: type:21;rtc_s:8674218.036285;mach:227470337392
2016-07-30 01:44:09 -0400 timed[62]: unc_ns:500930;src:13;before_mach:227469699175;after_mach:227469699346;rtc_s:8674218.009692624;type:11;t_ns:491550249399163221
2016-07-30 01:44:09 -0400 timed[62]: t_s:491550249.3991376;rateSf:0.9999892325502973;sf_var:6.033211989206314e-12;rtc_s:8674218.009692624;utc_var:3.237425987339789e-08;type:29;utc_sf_cov:8.287860067809633e-11
2016-07-30 01:44:09 -0400 timed[62]: _utc_sf_cov:7.340780371667235e-11;_rateSf:0.999989240065558;_rtc:8674218.009692624;_utc_var:2.867475192407557e-08;type:32;_utc:491550249.3991405;_sf_var:6.008966604059636e-12
2016-07-30 01:44:09 -0400 timed[62]: need_rtc_s:9174209.542882137;bb_want_rtc_s:inf;type:20;rtc_s:8674218.056411;want_rtc_s:8676709.542882139
2016-07-30 01:44:09 -0400 timed[62]: type:21;rtc_s:8674218.059051;mach:227470881741
2016-07-30 01:44:09 -0400 timed[62]: t_s:491550249.4298912;adjust:2.515316009521484e-05;didSet:1;mach:227470435165;rtc_s:8674218.040443666;unc_s:0.0001693536001689697;after_mach:227470435185;type:3;settimeofday:0
2016-07-30 01:44:10 -0400 timed[62]: type:21;rtc_s:8674219.176666001;mach:227497705265
2016-07-30 01:44:10 -0400 timed[62]: unc_ns:501039;src:13;before_mach:227496936195;after_mach:227496936332;rtc_s:8674219.144621417;type:11;t_ns:491550250534019749
2016-07-30 01:44:10 -0400 timed[62]: t_s:491550250.5340571;rateSf:0.999989240065558;sf_var:6.040517624497115e-12;rtc_s:8674219.144621417;utc_var:2.884913066048421e-08;type:29;utc_sf_cov:8.024545701084996e-11
2016-07-30 01:44:10 -0400 timed[62]: _utc_sf_cov:7.197431185924256e-11;_rateSf:0.9999892293679336;_rtc:8674219.144621417;_utc_var:2.587556236043341e-08;type:32;_utc:491550250.5340532;_sf_var:6.01751097550984e-12
2016-07-30 01:44:10 -0400 timed[62]: need_rtc_s:9174211.101680046;bb_want_rtc_s:inf;type:20;rtc_s:8674219.194746999;want_rtc_s:8676711.101680046
2016-07-30 01:44:10 -0400 timed[62]: type:21;rtc_s:8674219.197143;mach:227498197021
2016-07-30 01:44:10 -0400 timed[62]: t_s:491550250.5720073;adjust:-6.067752838134766e-05;didSet:1;mach:227497847411;rtc_s:8674219.182575917;unc_s:0.0001608783171244959;after_mach:227497847434;type:3;settimeofday:0
2016-07-30 01:51:12 -0400 timed[62]: type:21;rtc_s:8674640.796875;mach:237616638384
2016-07-30 01:51:12 -0400 timed[62]: unc_ns:500880;src:13;before_mach:237616060688;after_mach:237616060784;rtc_s:8674640.772804333;type:11;t_ns:491550672157557865
2016-07-30 01:51:12 -0400 timed[62]: t_s:491550672.1576949;rateSf:0.9999892293679336;sf_var:1.773877446057382e-11;rtc_s:8674640.772804333;utc_var:1.850867510187921e-06;type:29;utc_sf_cov:5.080134042483789e-09
2016-07-30 01:51:12 -0400 timed[62]: _utc_sf_cov:6.06404593916812e-10;_rateSf:0.9999888980062223;_rtc:8674640.772804333;_utc_var:2.209340445592994e-07;type:32;_utc:491550672.1575742;_sf_var:5.459589082134815e-12
2016-07-30 01:51:12 -0400 timed[62]: type:21;rtc_s:8674640.800720001;mach:237616730948
2016-07-30 01:51:12 -0400 timed[62]: need_rtc_s:9174617.270993521;bb_want_rtc_s:inf;type:20;rtc_s:8674640.808027999;want_rtc_s:8677117.270993521
2016-07-30 01:51:12 -0400 timed[62]: t_s:491550672.1827373;adjust:-0.006665706634521484;didSet:1;mach:237616664893;rtc_s:8674640.79796771;unc_s:0.0004700705855875417;after_mach:237616664899;type:3;settimeofday:0
2016-07-30 01:51:13 -0400 timed[62]: type:21;rtc_s:8674641.893463001;mach:237642959045
2016-07-30 01:51:13 -0400 timed[62]: unc_ns:500956;src:13;before_mach:237642290048;after_mach:237642290227;rtc_s:8674641.865588125;type:11;t_ns:491550673250434671
2016-07-30 01:51:13 -0400 timed[62]: t_s:491550673.2503459;rateSf:0.9999888980062223;sf_var:5.489968471550869e-12;rtc_s:8674641.865588125;utc_var:2.222659145877913e-07;type:29;utc_sf_cov:6.12387343428627e-10
2016-07-30 01:51:13 -0400 timed[62]: _utc_sf_cov:3.247581729920058e-10;_rateSf:0.9999890129343771;_rtc:8674641.865588125;_utc_var:1.178709408587591e-07;type:32;_utc:491550673.2503876;_sf_var:4.697492198691273e-12
2016-07-30 01:51:13 -0400 timed[62]: type:21;rtc_s:8674641.896728;mach:237643035092
2016-07-30 01:51:13 -0400 timed[62]: need_rtc_s:9174624.699419299;bb_want_rtc_s:inf;type:20;rtc_s:8674641.908919999;want_rtc_s:8677124.699419299
2016-07-30 01:51:13 -0400 timed[62]: t_s:491550673.2801186;adjust:-0.002357423305511475;didSet:1;mach:237643001287;rtc_s:8674641.895319458;unc_s:0.0003433569440593995;after_mach:237643001412;type:3;settimeofday:0
2016-07-30 01:51:14 -0400 timed[62]: type:21;rtc_s:8674642.962615;mach:237668616757
2016-07-30 01:51:14 -0400 timed[62]: unc_ns:500563;src:13;before_mach:237668419049;after_mach:237668419181;rtc_s:8674642.954377167;type:11;t_ns:491550674339124041
2016-07-30 01:51:14 -0400 timed[62]: t_s:491550674.3391647;rateSf:0.9999890129343771;sf_var:4.727760534061247e-12;rtc_s:8674642.954377167;utc_var:1.185837077960867e-07;type:29;utc_sf_cov:3.29889228939159e-10
2016-07-30 01:51:14 -0400 timed[62]: _utc_sf_cov:2.239167978235736e-10;_rateSf:0.9999889766071415;_rtc:8674642.954377167;_utc_var:8.049030339406192e-08;type:32;_utc:491550674.3391516;_sf_var:4.432954744638273e-12
2016-07-30 01:51:14 -0400 timed[62]: type:21;rtc_s:8674642.975341;mach:237668922505
2016-07-30 01:51:14 -0400 timed[62]: need_rtc_s:9174628.768970635;bb_want_rtc_s:inf;type:20;rtc_s:8674642.980361;want_rtc_s:8677128.768970635
2016-07-30 01:51:14 -0400 timed[62]: t_s:491550674.3502482;adjust:0.0001892447471618652;didSet:1;mach:237668685694;rtc_s:8674642.965473875;unc_s:0.0002837199224468212;after_mach:237668685715;type:3;settimeofday:0
2016-07-30 01:51:15 -0400 timed[62]: type:21;rtc_s:8674644.081634;mach:237695475967
2016-07-30 01:51:15 -0400 timed[62]: unc_ns:500949;src:13;before_mach:237694815635;after_mach:237694815807;rtc_s:8674644.054120166;type:11;t_ns:491550675438966144
2016-07-30 01:51:15 -0400 timed[62]: t_s:491550675.4388825;rateSf:0.9999889766071415;sf_var:4.463527600018823e-12;rtc_s:8674644.054120166;utc_var:8.098817895015112e-08;type:29;utc_sf_cov:2.288087199120415e-10
2016-07-30 01:51:15 -0400 timed[62]: _utc_sf_cov:1.729826872250801e-10;_rateSf:0.999989034292068;_rtc:8674644.054120166;_utc_var:6.122822956069308e-08;type:32;_utc:491550675.4389029;_sf_var:4.305807260551336e-12
2016-07-30 01:51:15 -0400 timed[62]: need_rtc_s:9174631.681950837;bb_want_rtc_s:inf;type:20;rtc_s:8674644.100753;want_rtc_s:8677131.681950837
2016-07-30 01:51:15 -0400 timed[62]: type:21;rtc_s:8674644.10321;mach:237695991516
2016-07-30 01:51:15 -0400 timed[62]: t_s:491550675.4710639;adjust:8.344650268554688e-07;didSet:0;mach:237695585231;rtc_s:8674644.086281458;unc_s:0.0002474687045663081;after_mach:237695585254;type:3;settimeofday:0
2016-07-30 01:51:15 -0400 timed[62]: type:21;rtc_s:8674644.176757;mach:237697756418
2016-07-30 01:51:15 -0400 timed[62]: t_s:491550675.5595762;adjust:1.013278961181641e-05;didSet:1;mach:237697709323;rtc_s:8674644.174794709;unc_s:0.0002475321002340788;after_mach:237697709346;type:3;settimeofday:0
2016-07-30 01:51:16 -0400 timed[62]: type:21;rtc_s:8674645.172454;mach:237721655608
2016-07-30 01:51:16 -0400 timed[62]: unc_ns:500915;src:13;before_mach:237721036157;after_mach:237721036330;rtc_s:8674645.146643542;type:11;t_ns:491550676531470616
2016-07-30 01:51:16 -0400 timed[62]: t_s:491550676.5314143;rateSf:0.999989034292068;sf_var:4.336179410390961e-12;rtc_s:8674645.146643542;utc_var:6.161135634663353e-08;type:29;utc_sf_cov:1.777034734495747e-10
2016-07-30 01:51:16 -0400 timed[62]: _utc_sf_cov:1.426711789454886e-10;_rateSf:0.9999890662854323;_rtc:8674645.146643542;_utc_var:4.946535189082347e-08;type:32;_utc:491550676.5314254;_sf_var:4.235136992475993e-12
2016-07-30 01:51:16 -0400 timed[62]: need_rtc_s:9174634.026239784;bb_want_rtc_s:inf;type:20;rtc_s:8674645.191543;want_rtc_s:8677134.026239784
2016-07-30 01:51:16 -0400 timed[62]: type:21;rtc_s:8674645.194;mach:237722170291
2016-07-30 01:51:16 -0400 timed[62]: t_s:491550676.5612787;adjust:-1.311302185058594e-06;didSet:0;mach:237721750222;rtc_s:8674645.176497126;unc_s:0.0002224313590880929;after_mach:237721750240;type:3;settimeofday:0
2016-07-30 01:51:16 -0400 timed[62]: type:21;rtc_s:8674645.262878001;mach:237723823496
2016-07-30 01:51:16 -0400 timed[62]: t_s:491550676.6472262;adjust:-7.808208465576172e-06;didSet:1;mach:237723813119;rtc_s:8674645.262445625;unc_s:0.0002224874042405516;after_mach:237723813138;type:3;settimeofday:0
2016-07-30 01:51:17 -0400 timed[62]: type:21;rtc_s:8674646.263731999;mach:237747846446
2016-07-30 01:51:17 -0400 timed[62]: unc_ns:500926;src:13;before_mach:237747213408;after_mach:237747213556;rtc_s:8674646.237355417;type:11;t_ns:491550677622171147
2016-07-30 01:51:17 -0400 timed[62]: t_s:491550677.6221254;rateSf:0.9999890662854323;sf_var:4.265458782597862e-12;rtc_s:8674646.237355417;utc_var:4.978162855397568e-08;type:29;utc_sf_cov:1.473070293232337e-10
2016-07-30 01:51:17 -0400 timed[62]: _utc_sf_cov:1.229207177675962e-10;_rateSf:0.9999890887097037;_rtc:8674646.237355417;_utc_var:4.154040402286095e-08;type:32;_utc:491550677.622133;_sf_var:4.193298123821565e-12
2016-07-30 01:51:17 -0400 timed[62]: type:21;rtc_s:8674646.269927001;mach:237747993946
2016-07-30 01:51:17 -0400 timed[62]: need_rtc_s:9174636.046623841;bb_want_rtc_s:inf;type:20;rtc_s:8674646.282516001;want_rtc_s:8677136.046623841
2016-07-30 01:51:17 -0400 timed[62]: t_s:491550677.6524997;adjust:-5.728006362915039e-05;didSet:1;mach:237747941038;rtc_s:8674646.2677225;unc_s:0.0002038360677104176;after_mach:237747941057;type:3;settimeofday:0
2016-07-30 01:51:18 -0400 timed[62]: type:21;rtc_s:8674647.380828001;mach:237774655858
2016-07-30 01:51:18 -0400 timed[62]: unc_ns:500936;src:13;before_mach:237774010839;after_mach:237774011005;rtc_s:8674647.353952209;type:11;t_ns:491550678738713974
2016-07-30 01:51:18 -0400 timed[62]: t_s:491550678.7387176;rateSf:0.9999890887097037;sf_var:4.224339514636197e-12;rtc_s:8674647.353952209;utc_var:4.182015083721628e-08;type:29;utc_sf_cov:1.276202713588071e-10
2016-07-30 01:51:18 -0400 timed[62]: _utc_sf_cov:1.093898308035699e-10;_rateSf:0.9999890871507152;_rtc:8674647.353952209;_utc_var:3.58461800429886e-08;type:32;_utc:491550678.738717;_sf_var:4.168706674388774e-12
2016-07-30 01:51:18 -0400 timed[62]: type:21;rtc_s:8674647.384948;mach:237774756942
2016-07-30 01:51:18 -0400 timed[62]: need_rtc_s:9174637.887408484;bb_want_rtc_s:inf;type:20;rtc_s:8674647.396499;want_rtc_s:8677137.887408484
2016-07-30 01:51:18 -0400 timed[62]: t_s:491550678.7671773;adjust:-0.0001166462898254395;didSet:1;mach:237774696098;rtc_s:8674647.382412834;unc_s:0.0001893493624671754;after_mach:237774696439;type:3;settimeofday:0
2016-07-30 01:51:19 -0400 timed[62]: type:21;rtc_s:8674648.444305001;mach:237800180717
2016-07-30 01:51:19 -0400 timed[62]: unc_ns:500953;src:13;before_mach:237799515019;after_mach:237799515199;rtc_s:8674648.416567584;type:11;t_ns:491550679801371988
2016-07-30 01:51:19 -0400 timed[62]: t_s:491550679.8013208;rateSf:0.9999890871507152;sf_var:4.198247381840624e-12;rtc_s:8674648.416567584;utc_var:3.608337689513786e-08;type:29;utc_sf_cov:1.138352578186444e-10
2016-07-30 01:51:19 -0400 timed[62]: _utc_sf_cov:9.952509674968907e-11;_rateSf:0.9999891074560715;_rtc:8674648.416567584;_utc_var:3.154735751787359e-08;type:32;_utc:491550679.8013272;_sf_var:4.153101915565532e-12
2016-07-30 01:51:19 -0400 timed[62]: type:21;rtc_s:8674648.4505;mach:237800328291
2016-07-30 01:51:19 -0400 timed[62]: need_rtc_s:9174639.535779681;bb_want_rtc_s:inf;type:20;rtc_s:8674648.46614;want_rtc_s:8677139.535779681
2016-07-30 01:51:19 -0400 timed[62]: t_s:491550679.8330414;adjust:7.241964340209961e-05;didSet:1;mach:237800275061;rtc_s:8674648.448282083;unc_s:0.0001776363989881384;after_mach:237800275079;type:3;settimeofday:0
2016-07-30 01:51:20 -0400 timed[62]: type:21;rtc_s:8674649.480957;mach:237825059932
2016-07-30 01:51:20 -0400 timed[62]: unc_ns:500938;src:13;before_mach:237824412201;after_mach:237824412378;rtc_s:8674649.453968208;type:11;t_ns:491550680838738234
2016-07-30 01:51:20 -0400 timed[62]: t_s:491550680.8387166;rateSf:0.9999891074560715;sf_var:4.181941652906713e-12;rtc_s:8674649.453968208;utc_var:3.175833222705643e-08;type:29;utc_sf_cov:1.038484864482868e-10
2016-07-30 01:51:20 -0400 timed[62]: _utc_sf_cov:9.218213274919325e-11;_rateSf:0.9999891154260828;_rtc:8674649.453968208;_utc_var:2.819059667957051e-08;type:32;_utc:491550680.838719;_sf_var:4.143793139144738e-12
2016-07-30 01:51:20 -0400 timed[62]: type:21;rtc_s:8674649.484283;mach:237825138878
2016-07-30 01:51:20 -0400 timed[62]: need_rtc_s:9174641.058940412;bb_want_rtc_s:inf;type:20;rtc_s:8674649.497237001;want_rtc_s:8677141.058940412
2016-07-30 01:51:20 -0400 timed[62]: t_s:491550680.868099;adjust:1.013278961181641e-06;didSet:0;mach:237825116451;rtc_s:8674649.483348543;unc_s:0.0001679201885822087;after_mach:237825116476;type:3;settimeofday:0
2016-07-30 01:51:20 -0400 timed[62]: type:21;rtc_s:8674649.537384;mach:237826411750
2016-07-30 01:51:20 -0400 timed[62]: t_s:491550680.9187441;adjust:6.407499313354492e-05;didSet:1;mach:237826330393;rtc_s:8674649.533994125;unc_s:0.0001679495859964901;after_mach:237826330422;type:3;settimeofday:0
2016-07-30 01:51:21 -0400 timed[62]: type:21;rtc_s:8674650.600646;mach:237851931159
2016-07-30 01:51:21 -0400 timed[62]: unc_ns:500931;src:13;before_mach:237851291705;after_mach:237851291882;rtc_s:8674650.574002083;type:11;t_ns:491550681958699844
2016-07-30 01:51:21 -0400 timed[62]: t_s:491550681.9587407;rateSf:0.9999891154260828;sf_var:4.174930080872734e-12;rtc_s:8674650.574002083;utc_var:2.840230221101939e-08;type:29;utc_sf_cov:9.68407586512258e-11
2016-07-30 01:51:21 -0400 timed[62]: _utc_sf_cov:8.699415864890478e-11;_rateSf:0.999989101250629;_rtc:8674650.574002083;_utc_var:2.551440549364475e-08;type:32;_utc:491550681.9587365;_sf_var:4.14135702023577e-12
2016-07-30 01:51:22 -0400 timed[62]: need_rtc_s:9174642.587387422;bb_want_rtc_s:inf;type:20;rtc_s:8674650.624647999;want_rtc_s:8677142.587387422
2016-07-30 01:51:22 -0400 timed[62]: type:21;rtc_s:8674650.626861;mach:237852559967
2016-07-30 01:51:22 -0400 timed[62]: t_s:491550681.9908182;adjust:-4.678964614868164e-05;didSet:1;mach:237852061322;rtc_s:8674650.606084125;unc_s:0.0001597549361748855;after_mach:237852061342;type:3;settimeofday:0
2016-07-30 01:52:55 -0400 timed[62]: type:21;rtc_s:8674743.717132;mach:240086737145
2016-07-30 01:52:55 -0400 timed[62]: type:9;src:10;rtc_s:8674743.714004001
2016-07-30 01:58:23 -0400 timed[62]: type:21;rtc_s:8675071.780578;mach:247960300251
2016-07-30 01:58:23 -0400 timed[62]: unc_ns:500951;src:13;before_mach:247959637560;after_mach:247959637682;rtc_s:8675071.752965875;type:11;t_ns:491551103133146074
2016-07-30 01:58:23 -0400 timed[62]: t_s:491551103.13311;rateSf:0.999989101250629;sf_var:1.585013221364166e-11;rtc_s:8675071.752965875;utc_var:1.525783656159916e-06;type:29;utc_sf_cov:4.296991518736408e-09
2016-07-30 01:58:23 -0400 timed[62]: _utc_sf_cov:6.069215505686105e-10;_rateSf:0.9999891884628047;_rtc:8675071.752965875;_utc_var:2.155068210842391e-07;type:32;_utc:491551103.133141;_sf_var:5.457964691892384e-12
2016-07-30 01:58:23 -0400 timed[62]: type:21;rtc_s:8675071.78363;mach:247960371425
2016-07-30 01:58:23 -0400 timed[62]: need_rtc_s:9175048.54160976;bb_want_rtc_s:inf;type:20;rtc_s:8675071.796645001;want_rtc_s:8677548.54160976
2016-07-30 01:58:23 -0400 timed[62]: t_s:491551103.1629292;adjust:-0.00660175085067749;didSet:1;mach:247960350412;rtc_s:8675071.782754458;unc_s:0.0004642705282475338;after_mach:247960350434;type:3;settimeofday:0
2016-07-30 01:58:24 -0400 timed[62]: type:21;rtc_s:8675072.884368001;mach:247986788846
2016-07-30 01:58:24 -0400 timed[62]: unc_ns:500937;src:13;before_mach:247986142891;after_mach:247986143058;rtc_s:8675072.857453208;type:11;t_ns:491551104237519265
2016-07-30 01:58:24 -0400 timed[62]: t_s:491551104.2376164;rateSf:0.9999891884628047;sf_var:5.488669439762203e-12;rtc_s:8675072.857453208;utc_var:2.168541660286679e-07;type:29;utc_sf_cov:6.129667599397546e-10
2016-07-30 01:58:24 -0400 timed[62]: _utc_sf_cov:3.288141904032485e-10;_rateSf:0.9999890612341292;_rtc:8675072.857453208;_utc_var:1.163272328915458e-07;type:32;_utc:491551104.2375714;_sf_var:4.685474910152281e-12
2016-07-30 01:58:24 -0400 timed[62]: type:21;rtc_s:8675072.887542;mach:247986867273
2016-07-30 01:58:24 -0400 timed[62]: need_rtc_s:9175055.804064037;bb_want_rtc_s:inf;type:20;rtc_s:8675072.90068;want_rtc_s:8677555.804064037
2016-07-30 01:58:24 -0400 timed[62]: t_s:491551104.2661134;adjust:-0.002704620361328125;didSet:1;mach:247986830159;rtc_s:8675072.885995584;unc_s:0.0003410992753771674;after_mach:247986830286;type:3;settimeofday:0
2016-07-30 01:58:25 -0400 timed[62]: type:21;rtc_s:8675073.983398;mach:248013167303
2016-07-30 01:58:25 -0400 timed[62]: unc_ns:500945;src:13;before_mach:248012510912;after_mach:248012511081;rtc_s:8675073.956048375;type:11;t_ns:491551105336169637
2016-07-30 01:58:25 -0400 timed[62]: t_s:491551105.3361545;rateSf:0.9999890612341292;sf_var:4.716015855783173e-12;rtc_s:8675073.956048375;utc_var:1.170553674916453e-07;type:29;utc_sf_cov:3.339784065602948e-10
2016-07-30 01:58:25 -0400 timed[62]: _utc_sf_cov:2.27745456641257e-10;_rateSf:0.9999890749739709;_rtc:8675073.956048375;_utc_var:7.982201123796922e-08;type:32;_utc:491551105.3361593;_sf_var:4.412915604180511e-12
2016-07-30 01:58:25 -0400 timed[62]: type:21;rtc_s:8675073.985564999;mach:248013218795
2016-07-30 01:58:25 -0400 timed[62]: need_rtc_s:9175059.829653645;bb_want_rtc_s:inf;type:20;rtc_s:8675073.999404;want_rtc_s:8677559.829653645
2016-07-30 01:58:25 -0400 timed[62]: t_s:491551105.3651947;adjust:0.0002786517143249512;didSet:1;mach:248013207251;rtc_s:8675073.985083999;unc_s:0.0002825552596713029;after_mach:248013207276;type:3;settimeofday:0
2016-07-30 01:58:26 -0400 timed[62]: type:21;rtc_s:8675075.036865;mach:248038448841
2016-07-30 01:58:26 -0400 timed[62]: unc_ns:500881;src:13;before_mach:248037869884;after_mach:248037870015;rtc_s:8675075.012741791;type:11;t_ns:491551106392774560
2016-07-30 01:58:26 -0400 timed[62]: t_s:491551106.3928412;rateSf:0.9999890749739709;sf_var:4.44229168114778e-12;rtc_s:8675075.012741791;utc_var:8.030826388677742e-08;type:29;utc_sf_cov:2.32424076259511e-10
2016-07-30 01:58:26 -0400 timed[62]: _utc_sf_cov:1.760650509666319e-10;_rateSf:0.9999890282085181;_rtc:8675075.012741791;_utc_var:6.083482744911452e-08;type:32;_utc:491551106.3928251;_sf_var:4.279180268217079e-12
2016-07-30 01:58:26 -0400 timed[62]: need_rtc_s:9175062.680383192;bb_want_rtc_s:inf;type:20;rtc_s:8675075.051955;want_rtc_s:8677562.680383192
2016-07-30 01:58:26 -0400 timed[62]: type:21;rtc_s:8675075.053526999;mach:248038847764
2016-07-30 01:58:26 -0400 timed[62]: t_s:491551106.4193423;adjust:6.127357482910156e-05;didSet:1;mach:248038505338;rtc_s:8675075.039259249;unc_s:0.0002466690233597617;after_mach:248038505347;type:3;settimeofday:0
2016-07-30 01:58:27 -0400 timed[62]: type:21;rtc_s:8675076.129547;mach:248064676114
2016-07-30 01:58:27 -0400 timed[62]: unc_ns:500929;src:13;before_mach:248064038985;after_mach:248064039160;rtc_s:8675076.102999959;type:11;t_ns:491551107483139094
2016-07-30 01:58:27 -0400 timed[62]: t_s:491551107.4830713;rateSf:0.9999890282085181;sf_var:4.309489445289053e-12;rtc_s:8675076.102999959;utc_var:6.122383868080879e-08;type:29;utc_sf_cov:1.807469846205771e-10
2016-07-30 01:58:27 -0400 timed[62]: _utc_sf_cov:1.452965294363431e-10;_rateSf:0.9999890674841903;_rtc:8675076.102999959;_utc_var:4.921582120866728e-08;type:32;_utc:491551107.4830846;_sf_var:4.204831471923759e-12
2016-07-30 01:58:27 -0400 timed[62]: need_rtc_s:9175065.010680407;bb_want_rtc_s:inf;type:20;rtc_s:8675076.149336999;want_rtc_s:8677565.010680407
2016-07-30 01:58:27 -0400 timed[62]: type:21;rtc_s:8675076.151823999;mach:248065207951
2016-07-30 01:58:27 -0400 timed[62]: t_s:491551107.5138307;adjust:-4.231929779052734e-05;didSet:1;mach:248064774088;rtc_s:8675076.133746374;unc_s:0.0002218731012171998;after_mach:248064774116;type:3;settimeofday:0
2016-07-30 01:58:28 -0400 timed[62]: type:21;rtc_s:8675077.169097001;mach:248089625010
2016-07-30 01:58:28 -0400 timed[62]: unc_ns:500930;src:13;before_mach:248088986754;after_mach:248088986940;rtc_s:8675077.142503001;type:11;t_ns:491551108522613655
2016-07-30 01:58:28 -0400 timed[62]: t_s:491551108.5225762;rateSf:0.9999890674841903;sf_var:4.233729656481763e-12;rtc_s:8675077.142503001;utc_var:4.952244758638953e-08;type:29;utc_sf_cov:1.49682484416427e-10
2016-07-30 01:58:28 -0400 timed[62]: _utc_sf_cov:1.250110265566818e-10;_rateSf:0.9999890861321988;_rtc:8675077.142503001;_utc_var:4.135989614623611e-08;type:32;_utc:491551108.5225824;_sf_var:4.159159733414455e-12
2016-07-30 01:58:28 -0400 timed[62]: type:21;rtc_s:8675077.17633;mach:248089797128
2016-07-30 01:58:28 -0400 timed[62]: need_rtc_s:9175066.973936712;bb_want_rtc_s:inf;type:20;rtc_s:8675077.186232999;want_rtc_s:8677566.973936712
2016-07-30 01:58:28 -0400 timed[62]: t_s:491551108.55371;adjust:-4.89354133605957e-05;didSet:1;mach:248089732352;rtc_s:8675077.173630999;unc_s:0.0002033928726639199;after_mach:248089732371;type:3;settimeofday:0
2016-07-30 01:58:29 -0400 timed[62]: type:21;rtc_s:8675078.263823999;mach:248115898848
2016-07-30 01:58:29 -0400 timed[62]: unc_ns:500949;src:13;before_mach:248115238821;after_mach:248115238991;rtc_s:8675078.236322874;type:11;t_ns:491551109616431877
2016-07-30 01:58:29 -0400 timed[62]: t_s:491551109.6163903;rateSf:0.9999890861321988;sf_var:4.189567925895183e-12;rtc_s:8675078.236322874;utc_var:4.163836355697183e-08;type:29;utc_sf_cov:1.295770286723916e-10
2016-07-30 01:58:29 -0400 timed[62]: _utc_sf_cov:1.111368665957301e-10;_rateSf:0.9999891045307643;_rtc:8675078.236322874;_utc_var:3.571279032486149e-08;type:32;_utc:491551109.6163962;_sf_var:4.132182831813333e-12
2016-07-30 01:58:29 -0400 timed[62]: type:21;rtc_s:8675078.267059;mach:248115976829
2016-07-30 01:58:29 -0400 timed[62]: need_rtc_s:9175068.787408873;bb_want_rtc_s:inf;type:20;rtc_s:8675078.279586;want_rtc_s:8677568.787408872
2016-07-30 01:58:29 -0400 timed[62]: t_s:491551109.6456165;adjust:-9.85264778137207e-05;didSet:1;mach:248115940455;rtc_s:8675078.265543416;unc_s:0.0001889983539365245;after_mach:248115940474;type:3;settimeofday:0
2016-07-30 01:58:30 -0400 timed[62]: type:21;rtc_s:8675079.378417;mach:248142647033
2016-07-30 01:58:30 -0400 timed[62]: unc_ns:500925;src:13;before_mach:248142014798;after_mach:248142014966;rtc_s:8675079.352073874;type:11;t_ns:491551110732080008
2016-07-30 01:58:30 -0400 timed[62]: t_s:491551110.7321351;rateSf:0.9999891045307643;sf_var:4.163200709616703e-12;rtc_s:8675079.352073874;utc_var:3.596594949198226e-08;type:29;utc_sf_cov:1.157646578371e-10
2016-07-30 01:58:30 -0400 timed[62]: _utc_sf_cov:1.012519627301036e-10;_rateSf:0.9999890822834097;_rtc:8675079.352073874;_utc_var:3.145712210923079e-08;type:32;_utc:491551110.7321282;_sf_var:4.116488271683426e-12
2016-07-30 01:58:30 -0400 timed[62]: type:21;rtc_s:8675079.385101;mach:248142806702
2016-07-30 01:58:30 -0400 timed[62]: need_rtc_s:9175070.483995995;bb_want_rtc_s:inf;type:20;rtc_s:8675079.399916999;want_rtc_s:8677570.483995995
2016-07-30 01:58:30 -0400 timed[62]: t_s:491551110.762586;adjust:0.0001109838485717773;didSet:1;mach:248142745047;rtc_s:8675079.382532042;unc_s:0.0001774064281160901;after_mach:248142745070;type:3;settimeofday:0
2016-07-30 01:58:31 -0400 timed[62]: type:21;rtc_s:8675080.472259;mach:248168901688
2016-07-30 01:58:31 -0400 timed[62]: unc_ns:500954;src:13;before_mach:248168235572;after_mach:248168235734;rtc_s:8675080.444504166;type:11;t_ns:491551111824594744
2016-07-30 01:58:31 -0400 timed[62]: t_s:491551111.8245466;rateSf:0.9999890822834097;sf_var:4.146857833792613e-12;rtc_s:8675080.444504166;utc_var:3.168326824597438e-08;type:29;utc_sf_cov:1.057655275283044e-10
2016-07-30 01:58:31 -0400 timed[62]: _utc_sf_cov:9.390939241238694e-11;_rateSf:0.9999891003054779;_rtc:8675080.444504166;_utc_var:2.813162795242409e-08;type:32;_utc:491551111.824552;_sf_var:4.107279518347166e-12
2016-07-30 01:58:31 -0400 timed[62]: type:21;rtc_s:8675080.480102001;mach:248169089150
2016-07-30 01:58:31 -0400 timed[62]: need_rtc_s:9175072.058261272;bb_want_rtc_s:inf;type:20;rtc_s:8675080.488754001;want_rtc_s:8677572.058261272
2016-07-30 01:58:31 -0400 timed[62]: t_s:491551111.8540602;adjust:-0.000105738639831543;didSet:1;mach:248168943008;rtc_s:8675080.474012751;unc_s:0.0001677449422189355;after_mach:248168943028;type:3;settimeofday:0
2016-07-30 01:58:32 -0400 timed[62]: type:21;rtc_s:8675081.588927999;mach:248195702040
2016-07-30 01:58:32 -0400 timed[62]: unc_ns:500960;src:13;before_mach:248195028251;after_mach:248195028419;rtc_s:8675081.560853457;type:11;t_ns:491551112940939223
2016-07-30 01:58:32 -0400 timed[62]: t_s:491551112.9408891;rateSf:0.9999891003054779;sf_var:4.138314028638534e-12;rtc_s:8675081.560853457;utc_var:2.834643085034406e-08;type:29;utc_sf_cov:9.851187366764528e-11
2016-07-30 01:58:32 -0400 timed[62]: _utc_sf_cov:8.851408781027762e-11;_rateSf:0.9999891179854489;_rtc:8675081.560853457;_utc_var:2.546960458655227e-08;type:32;_utc:491551112.9408942;_sf_var:4.103568891850873e-12
2016-07-30 01:58:32 -0400 timed[62]: type:21;rtc_s:8675081.59317;mach:248195802659
2016-07-30 01:58:32 -0400 timed[62]: need_rtc_s:9175073.581253752;bb_want_rtc_s:inf;type:20;rtc_s:8675081.605697;want_rtc_s:8677573.58125375
2016-07-30 01:58:33 -0400 timed[62]: t_s:491551112.9711252;adjust:-3.755092620849609e-06;didSet:1;mach:248195752615;rtc_s:8675081.591084834;unc_s:0.0001596124439668572;after_mach:248195752640;type:3;settimeofday:0
2016-07-30 01:59:04 -0400 timed[62]: type:21;rtc_s:8675112.816345001;mach:248945160465
2016-07-30 01:59:04 -0400 timed[62]: unc_ns:500937;src:13;before_mach:248944513841;after_mach:248944513992;rtc_s:8675112.789402334;type:11;t_ns:491551144169013128
2016-07-30 01:59:04 -0400 timed[62]: t_s:491551144.1691032;rateSf:0.9999891179854489;sf_var:4.971722550629838e-12;rtc_s:8675112.789402334;utc_var:3.528204362979179e-08;type:29;utc_sf_cov:2.302181790022752e-10
2016-07-30 01:59:04 -0400 timed[62]: _utc_sf_cov:2.01839498969516e-10;_rateSf:0.999989045544684;_rtc:8675112.789402334;_utc_var:3.093287437039193e-08;type:32;_utc:491551144.1690921;_sf_var:4.786549375623802e-12
2016-07-30 01:59:04 -0400 timed[62]: type:21;rtc_s:8675112.820219999;mach:248945255631
2016-07-30 01:59:04 -0400 timed[62]: need_rtc_s:9175103.995530268;bb_want_rtc_s:inf;type:20;rtc_s:8675112.832381001;want_rtc_s:8677603.995530268
2016-07-30 01:59:04 -0400 timed[62]: t_s:491551144.1978766;adjust:-0.000508427619934082;didSet:1;mach:248945206842;rtc_s:8675112.818187125;unc_s:0.0001759132946311913;after_mach:248945206866;type:3;settimeofday:0
2016-07-30 01:59:05 -0400 timed[62]: type:21;rtc_s:8675113.879608;mach:248970681656
2016-07-30 01:59:05 -0400 timed[62]: unc_ns:500956;src:13;before_mach:248970013216;after_mach:248970013369;rtc_s:8675113.851756332;type:11;t_ns:491551145231470410
2016-07-30 01:59:05 -0400 timed[62]: t_s:491551145.2314345;rateSf:0.999989045544684;sf_var:4.816082816779951e-12;rtc_s:8675113.851756332;utc_var:3.136713755893818e-08;type:29;utc_sf_cov:2.06940196322007e-10
2016-07-30 01:59:05 -0400 timed[62]: _utc_sf_cov:1.83948457565503e-10;_rateSf:0.99998907188945;_rtc:8675113.851756332;_utc_var:2.788214505814709e